declare -x AZ_BATCH_ACCOUNT_NAME="exp1hildebatch"
declare -x AZ_BATCH_ACCOUNT_URL="https://exp1hildebatch.southeastasia.batch.azure.com/"
declare -x AZ_BATCH_CERTIFICATES_DIR="/mnt/batch/tasks/workitems/svm/job-1/task-00045/certs"
declare -x AZ_BATCH_JOB_ID="svm"
declare -x AZ_BATCH_JOB_PREP_DIR="/mnt/batch/tasks/workitems/svm/job-1/jobpreparation"
declare -x AZ_BATCH_JOB_PREP_WORKING_DIR="/mnt/batch/tasks/workitems/svm/job-1/jobpreparation/wd"
declare -x AZ_BATCH_NODE_ID="tvm-2112069986_10-20180411t200903z"
declare -x AZ_BATCH_NODE_IS_DEDICATED="true"
declare -x AZ_BATCH_NODE_ROOT_DIR="/mnt/batch/tasks"
declare -x AZ_BATCH_NODE_SHARED_DIR="/mnt/batch/tasks/shared"
declare -x AZ_BATCH_NODE_STARTUP_DIR="/mnt/batch/tasks/startup"
declare -x AZ_BATCH_POOL_ID="experiment"
declare -x AZ_BATCH_TASK_DIR="/mnt/batch/tasks/workitems/svm/job-1/task-00045"
declare -x AZ_BATCH_TASK_ID="task-00045"
declare -x AZ_BATCH_TASK_USER="_azbatch"
declare -x AZ_BATCH_TASK_USER_IDENTITY="PoolAdmin"
declare -x AZ_BATCH_TASK_WORKING_DIR="/mnt/batch/tasks/workitems/svm/job-1/task-00045/wd"
declare -x HOME="/mnt/batch/tasks/workitems/svm/job-1/task-00045/wd"
declare -x OLDPWD
declare -x PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/mnt/batch/tasks/shared:/mnt/batch/tasks/workitems/svm/job-1/task-00045/wd"
declare -x PWD="/mnt/batch/tasks/workitems/svm/job-1/task-00045/wd"
declare -x SHLVL="1"
declare -x USER="_azbatch"
[2018-04-11 20:42:03] Results will not be logged locally.
[2018-04-11 20:42:03] Retrieving tasks...
[2018-04-11 20:42:03] Tasks will be retrieved based on provided task ids.
[2018-04-11 20:42:03] The following tasks were retrieved: [146822].
[2018-04-11 20:42:10] Downloaded task 146822.
[2018-04-11 20:42:10] Prepared pipelines.
[2018-04-11 20:42:10] Started run 1 on task 146822, dataset 'segment'.
[2018-04-11 20:42:10] Started run 7 on task 146822, dataset 'segment'.
[2018-04-11 20:42:10] Started run 3 on task 146822, dataset 'segment'.
[2018-04-11 20:42:10] Started run 8 on task 146822, dataset 'segment'.
[2018-04-11 20:42:10] Started run 2 on task 146822, dataset 'segment'.
[2018-04-11 20:42:10] Started run 4 on task 146822, dataset 'segment'.
[2018-04-11 20:42:10] Started run 5 on task 146822, dataset 'segment'.
[2018-04-11 20:42:10] Started run 6 on task 146822, dataset 'segment'.
[2018-04-11 20:42:15] Error in run 2: Run already exists in server. Run id(s): {8959917}
[2018-04-11 20:42:15] Error in run 1: Run already exists in server. Run id(s): {8941375}
[2018-04-11 20:42:15] Started run 9 on task 146822, dataset 'segment'.
[2018-04-11 20:42:15] Started run 10 on task 146822, dataset 'segment'.
[2018-04-11 20:42:15] Error in run 8: Run already exists in server. Run id(s): {8959923}
[2018-04-11 20:42:15] Started run 11 on task 146822, dataset 'segment'.
[2018-04-11 20:42:15] Error in run 3: Run already exists in server. Run id(s): {8959915}
[2018-04-11 20:42:15] Error in run 4: Run already exists in server. Run id(s): {8959918}
[2018-04-11 20:42:15] Started run 12 on task 146822, dataset 'segment'.
[2018-04-11 20:42:15] Error in run 7: Run already exists in server. Run id(s): {8959921}
[2018-04-11 20:42:15] Started run 14 on task 146822, dataset 'segment'.
[2018-04-11 20:42:15] Started run 13 on task 146822, dataset 'segment'.
[2018-04-11 20:42:15] Error in run 5: Run already exists in server. Run id(s): {8959919}
[2018-04-11 20:42:15] Started run 15 on task 146822, dataset 'segment'.
[2018-04-11 20:42:15] Error in run 6: Run already exists in server. Run id(s): {8959922}
[2018-04-11 20:42:15] Started run 16 on task 146822, dataset 'segment'.
[2018-04-11 20:42:19] Error in run 10: Run already exists in server. Run id(s): {8959928}
[2018-04-11 20:42:19] Error in run 12: Run already exists in server. Run id(s): {8959930}
[2018-04-11 20:42:19] Error in run 11: Run already exists in server. Run id(s): {8959931}
[2018-04-11 20:42:19] Started run 17 on task 146822, dataset 'segment'.
[2018-04-11 20:42:19] Error in run 9: Run already exists in server. Run id(s): {8959924}
[2018-04-11 20:42:19] Started run 20 on task 146822, dataset 'segment'.
[2018-04-11 20:42:19] Started run 19 on task 146822, dataset 'segment'.
[2018-04-11 20:42:19] Error in run 14: Run already exists in server. Run id(s): {8959934}
[2018-04-11 20:42:19] Error in run 13: Run already exists in server. Run id(s): {8959932}
[2018-04-11 20:42:19] Started run 21 on task 146822, dataset 'segment'.
[2018-04-11 20:42:19] Started run 22 on task 146822, dataset 'segment'.
[2018-04-11 20:42:19] Started run 18 on task 146822, dataset 'segment'.
[2018-04-11 20:42:19] Error in run 16: Run already exists in server. Run id(s): {8959936}
[2018-04-11 20:42:19] Started run 23 on task 146822, dataset 'segment'.
[2018-04-11 20:42:19] Error in run 15: Run already exists in server. Run id(s): {8959935}
[2018-04-11 20:42:19] Started run 24 on task 146822, dataset 'segment'.
[2018-04-11 20:42:24] Error in run 17: Run already exists in server. Run id(s): {8959937}
[2018-04-11 20:42:24] Started run 25 on task 146822, dataset 'segment'.
[2018-04-11 20:42:24] Error in run 20: Run already exists in server. Run id(s): {8959943}
[2018-04-11 20:42:24] Started run 26 on task 146822, dataset 'segment'.
[2018-04-11 20:42:24] Error in run 18: Run already exists in server. Run id(s): {8959941}
[2018-04-11 20:42:24] Started run 27 on task 146822, dataset 'segment'.
[2018-04-11 20:42:24] Error in run 22: Run already exists in server. Run id(s): {8959947}
[2018-04-11 20:42:24] Started run 28 on task 146822, dataset 'segment'.
[2018-04-11 20:42:24] Error in run 21: Run already exists in server. Run id(s): {8959944}
[2018-04-11 20:42:24] Error in run 19: Run already exists in server. Run id(s): {8959940}
[2018-04-11 20:42:24] Started run 29 on task 146822, dataset 'segment'.
[2018-04-11 20:42:24] Started run 30 on task 146822, dataset 'segment'.
[2018-04-11 20:42:24] Error in run 23: Run already exists in server. Run id(s): {8959946}
[2018-04-11 20:42:24] Started run 31 on task 146822, dataset 'segment'.
[2018-04-11 20:42:24] Error in run 24: Run already exists in server. Run id(s): {8959948}
[2018-04-11 20:42:24] Started run 32 on task 146822, dataset 'segment'.
[2018-04-11 20:42:28] Error in run 26: Run already exists in server. Run id(s): {8959954}
[2018-04-11 20:42:28] Started run 33 on task 146822, dataset 'segment'.
[2018-04-11 20:42:28] Error in run 25: Run already exists in server. Run id(s): {8959949}
[2018-04-11 20:42:28] Started run 34 on task 146822, dataset 'segment'.
[2018-04-11 20:42:29] Error in run 27: Run already exists in server. Run id(s): {8959955}
[2018-04-11 20:42:29] Error in run 30: Run already exists in server. Run id(s): {8959958}
[2018-04-11 20:42:29] Error in run 28: Run already exists in server. Run id(s): {8959952}
[2018-04-11 20:42:29] Started run 35 on task 146822, dataset 'segment'.
[2018-04-11 20:42:29] Error in run 29: Run already exists in server. Run id(s): {8959953}
[2018-04-11 20:42:29] Started run 38 on task 146822, dataset 'segment'.
[2018-04-11 20:42:29] Started run 36 on task 146822, dataset 'segment'.
[2018-04-11 20:42:29] Started run 37 on task 146822, dataset 'segment'.
[2018-04-11 20:42:29] Error in run 31: Run already exists in server. Run id(s): {8959959}
[2018-04-11 20:42:29] Started run 39 on task 146822, dataset 'segment'.
[2018-04-11 20:42:29] Error in run 32: Run already exists in server. Run id(s): {8959960}
[2018-04-11 20:42:29] Started run 40 on task 146822, dataset 'segment'.
[2018-04-11 20:42:33] Error in run 33: Run already exists in server. Run id(s): {8959961}
[2018-04-11 20:42:33] Started run 41 on task 146822, dataset 'segment'.
[2018-04-11 20:42:33] Error in run 34: Run already exists in server. Run id(s): {8959967}
[2018-04-11 20:42:33] Started run 42 on task 146822, dataset 'segment'.
[2018-04-11 20:42:33] Error in run 38: Run already exists in server. Run id(s): {8959970}
[2018-04-11 20:42:33] Error in run 36: Run already exists in server. Run id(s): {8959964}
[2018-04-11 20:42:33] Error in run 37: Run already exists in server. Run id(s): {8959966}
[2018-04-11 20:42:33] Error in run 39: Run already exists in server. Run id(s): {8959971}
[2018-04-11 20:42:33] Error in run 40: Run already exists in server. Run id(s): {8959973}
[2018-04-11 20:42:33] Started run 43 on task 146822, dataset 'segment'.
[2018-04-11 20:42:33] Started run 46 on task 146822, dataset 'segment'.
[2018-04-11 20:42:33] Error in run 35: Run already exists in server. Run id(s): {8959963}
[2018-04-11 20:42:33] Started run 44 on task 146822, dataset 'segment'.
[2018-04-11 20:42:33] Started run 48 on task 146822, dataset 'segment'.
[2018-04-11 20:42:33] Started run 45 on task 146822, dataset 'segment'.
[2018-04-11 20:42:33] Started run 47 on task 146822, dataset 'segment'.
[2018-04-11 20:42:37] Error in run 42: Run already exists in server. Run id(s): {8959975}
[2018-04-11 20:42:37] Started run 49 on task 146822, dataset 'segment'.
[2018-04-11 20:42:37] Error in run 41: Run already exists in server. Run id(s): {8959972}
[2018-04-11 20:42:37] Started run 50 on task 146822, dataset 'segment'.
[2018-04-11 20:42:38] Error in run 46: Run already exists in server. Run id(s): {8959982}
[2018-04-11 20:42:38] Started run 51 on task 146822, dataset 'segment'.
[2018-04-11 20:42:38] Error in run 44: Run already exists in server. Run id(s): {8959977}
[2018-04-11 20:42:38] Error in run 43: Run already exists in server. Run id(s): {8959976}
[2018-04-11 20:42:38] Started run 52 on task 146822, dataset 'segment'.
[2018-04-11 20:42:38] Error in run 45: Run already exists in server. Run id(s): {8959979}
[2018-04-11 20:42:38] Started run 54 on task 146822, dataset 'segment'.
[2018-04-11 20:42:38] Error in run 48: Run already exists in server. Run id(s): {8959984}
[2018-04-11 20:42:38] Started run 55 on task 146822, dataset 'segment'.
[2018-04-11 20:42:38] Error in run 47: Run already exists in server. Run id(s): {8959983}
[2018-04-11 20:42:38] Started run 56 on task 146822, dataset 'segment'.
[2018-04-11 20:42:38] Started run 53 on task 146822, dataset 'segment'.
[2018-04-11 20:42:42] Error in run 49: Run already exists in server. Run id(s): {8959985}
[2018-04-11 20:42:42] Started run 57 on task 146822, dataset 'segment'.
[2018-04-11 20:42:42] Error in run 50: Run already exists in server. Run id(s): {8959986}
[2018-04-11 20:42:42] Started run 58 on task 146822, dataset 'segment'.
[2018-04-11 20:42:43] Error in run 56: Run already exists in server. Run id(s): {8959996}
[2018-04-11 20:42:43] Error in run 51: Run already exists in server. Run id(s): {8959988}
[2018-04-11 20:42:43] Error in run 54: Run already exists in server. Run id(s): {8959994}
[2018-04-11 20:42:43] Error in run 52: Run already exists in server. Run id(s): {8959990}
[2018-04-11 20:42:43] Started run 59 on task 146822, dataset 'segment'.
[2018-04-11 20:42:43] Error in run 53: Run already exists in server. Run id(s): {8959991}
[2018-04-11 20:42:43] Started run 63 on task 146822, dataset 'segment'.
[2018-04-11 20:42:43] Error in run 55: Run already exists in server. Run id(s): {8959995}
[2018-04-11 20:42:43] Started run 60 on task 146822, dataset 'segment'.
[2018-04-11 20:42:43] Started run 64 on task 146822, dataset 'segment'.
[2018-04-11 20:42:43] Started run 61 on task 146822, dataset 'segment'.
[2018-04-11 20:42:43] Started run 62 on task 146822, dataset 'segment'.
[2018-04-11 20:42:47] Error in run 57: Run already exists in server. Run id(s): {8959997}
[2018-04-11 20:42:47] Error in run 58: Run already exists in server. Run id(s): {8959999}
[2018-04-11 20:42:47] Started run 65 on task 146822, dataset 'segment'.
[2018-04-11 20:42:47] Started run 66 on task 146822, dataset 'segment'.
[2018-04-11 20:42:48] Error in run 61: Run already exists in server. Run id(s): {8960003}
[2018-04-11 20:42:48] Started run 67 on task 146822, dataset 'segment'.
[2018-04-11 20:42:48] Error in run 60: Run already exists in server. Run id(s): {8960000}
[2018-04-11 20:42:48] Started run 68 on task 146822, dataset 'segment'.
[2018-04-11 20:42:48] Error in run 59: Run already exists in server. Run id(s): {8960001}
[2018-04-11 20:42:48] Error in run 64: Run already exists in server. Run id(s): {8960007}
[2018-04-11 20:42:48] Error in run 63: Run already exists in server. Run id(s): {8960008}
[2018-04-11 20:42:48] Started run 69 on task 146822, dataset 'segment'.
[2018-04-11 20:42:48] Error in run 62: Run already exists in server. Run id(s): {8960006}
[2018-04-11 20:42:48] Started run 72 on task 146822, dataset 'segment'.
[2018-04-11 20:42:48] Started run 70 on task 146822, dataset 'segment'.
[2018-04-11 20:42:48] Started run 71 on task 146822, dataset 'segment'.
[2018-04-11 20:42:51] Error in run 66: Run already exists in server. Run id(s): {8960010}
[2018-04-11 20:42:51] Error in run 65: Run already exists in server. Run id(s): {8960009}
[2018-04-11 20:42:51] Started run 73 on task 146822, dataset 'segment'.
[2018-04-11 20:42:51] Started run 74 on task 146822, dataset 'segment'.
[2018-04-11 20:42:52] Error in run 72: Run already exists in server. Run id(s): {8960020}
[2018-04-11 20:42:52] Started run 75 on task 146822, dataset 'segment'.
[2018-04-11 20:42:52] Error in run 69: Run already exists in server. Run id(s): {8960014}
[2018-04-11 20:42:52] Started run 76 on task 146822, dataset 'segment'.
[2018-04-11 20:42:52] Error in run 68: Run already exists in server. Run id(s): {8960012}
[2018-04-11 20:42:52] Error in run 67: Run already exists in server. Run id(s): {8960011}
[2018-04-11 20:42:52] Started run 77 on task 146822, dataset 'segment'.
[2018-04-11 20:42:52] Error in run 70: Run already exists in server. Run id(s): {8960019}
[2018-04-11 20:42:52] Started run 79 on task 146822, dataset 'segment'.
[2018-04-11 20:42:52] Started run 78 on task 146822, dataset 'segment'.
[2018-04-11 20:42:52] Error in run 71: Run already exists in server. Run id(s): {8960018}
[2018-04-11 20:42:52] Started run 80 on task 146822, dataset 'segment'.
[2018-04-11 20:42:56] Error in run 74: Run already exists in server. Run id(s): {8960023}
[2018-04-11 20:42:56] Error in run 73: Run already exists in server. Run id(s): {8960021}
[2018-04-11 20:42:56] Started run 81 on task 146822, dataset 'segment'.
[2018-04-11 20:42:56] Started run 82 on task 146822, dataset 'segment'.
[2018-04-11 20:42:57] Error in run 76: Run already exists in server. Run id(s): {8960025}
[2018-04-11 20:42:57] Started run 83 on task 146822, dataset 'segment'.
[2018-04-11 20:42:57] Error in run 75: Run already exists in server. Run id(s): {8960022}
[2018-04-11 20:42:57] Error in run 79: Run already exists in server. Run id(s): {8960030}
[2018-04-11 20:42:57] Started run 84 on task 146822, dataset 'segment'.
[2018-04-11 20:42:57] Started run 85 on task 146822, dataset 'segment'.
[2018-04-11 20:42:57] Error in run 77: Run already exists in server. Run id(s): {8960026}
[2018-04-11 20:42:57] Started run 86 on task 146822, dataset 'segment'.
[2018-04-11 20:42:57] Error in run 80: Run already exists in server. Run id(s): {8960032}
[2018-04-11 20:42:57] Started run 87 on task 146822, dataset 'segment'.
[2018-04-11 20:42:57] Error in run 78: Run already exists in server. Run id(s): {8960031}
[2018-04-11 20:42:57] Started run 88 on task 146822, dataset 'segment'.
[2018-04-11 20:43:01] Error in run 81: Run already exists in server. Run id(s): {8960033}
[2018-04-11 20:43:01] Started run 89 on task 146822, dataset 'segment'.
[2018-04-11 20:43:01] Error in run 82: Run already exists in server. Run id(s): {8960034}
[2018-04-11 20:43:01] Started run 90 on task 146822, dataset 'segment'.
[2018-04-11 20:43:01] Error in run 86: Run already exists in server. Run id(s): {8960041}
[2018-04-11 20:43:01] Error in run 84: Run already exists in server. Run id(s): {8960036}
[2018-04-11 20:43:01] Error in run 88: Run already exists in server. Run id(s): {8960045}
[2018-04-11 20:43:01] Started run 91 on task 146822, dataset 'segment'.
[2018-04-11 20:43:01] Started run 93 on task 146822, dataset 'segment'.
[2018-04-11 20:43:01] Started run 92 on task 146822, dataset 'segment'.
[2018-04-11 20:43:01] Error in run 87: Run already exists in server. Run id(s): {8960043}
[2018-04-11 20:43:01] Error in run 85: Run already exists in server. Run id(s): {8960038}
[2018-04-11 20:43:01] Error in run 83: Run already exists in server. Run id(s): {8960035}
[2018-04-11 20:43:01] Started run 94 on task 146822, dataset 'segment'.
[2018-04-11 20:43:01] Started run 95 on task 146822, dataset 'segment'.
[2018-04-11 20:43:01] Started run 96 on task 146822, dataset 'segment'.
[2018-04-11 20:43:05] Error in run 89: Run already exists in server. Run id(s): {8960044}
[2018-04-11 20:43:05] Error in run 90: Run already exists in server. Run id(s): {8960046}
[2018-04-11 20:43:05] Started run 97 on task 146822, dataset 'segment'.
[2018-04-11 20:43:05] Started run 98 on task 146822, dataset 'segment'.
[2018-04-11 20:43:06] Error in run 96: Run already exists in server. Run id(s): {8960056}
[2018-04-11 20:43:06] Error in run 92: Run already exists in server. Run id(s): {8960050}
[2018-04-11 20:43:06] Started run 99 on task 146822, dataset 'segment'.
[2018-04-11 20:43:06] Error in run 91: Run already exists in server. Run id(s): {8960047}
[2018-04-11 20:43:06] Started run 101 on task 146822, dataset 'segment'.
[2018-04-11 20:43:06] Started run 100 on task 146822, dataset 'segment'.
[2018-04-11 20:43:06] Error in run 93: Run already exists in server. Run id(s): {8960049}
[2018-04-11 20:43:06] Error in run 94: Run already exists in server. Run id(s): {8960053}
[2018-04-11 20:43:06] Started run 102 on task 146822, dataset 'segment'.
[2018-04-11 20:43:06] Error in run 95: Run already exists in server. Run id(s): {8960054}
[2018-04-11 20:43:06] Started run 104 on task 146822, dataset 'segment'.
[2018-04-11 20:43:06] Started run 103 on task 146822, dataset 'segment'.
[2018-04-11 20:43:10] Error in run 97: Run already exists in server. Run id(s): {8960057}
[2018-04-11 20:43:10] Started run 105 on task 146822, dataset 'segment'.
[2018-04-11 20:43:10] Error in run 98: Run already exists in server. Run id(s): {8960059}
[2018-04-11 20:43:10] Started run 106 on task 146822, dataset 'segment'.
[2018-04-11 20:43:11] Error in run 99: Run already exists in server. Run id(s): {8960058}
[2018-04-11 20:43:11] Error in run 102: Run already exists in server. Run id(s): {8960065}
[2018-04-11 20:43:11] Error in run 101: Run already exists in server. Run id(s): {8960062}
[2018-04-11 20:43:11] Started run 107 on task 146822, dataset 'segment'.
[2018-04-11 20:43:11] Error in run 103: Run already exists in server. Run id(s): {8960067}
[2018-04-11 20:43:11] Error in run 104: Run already exists in server. Run id(s): {8960068}
[2018-04-11 20:43:11] Started run 109 on task 146822, dataset 'segment'.
[2018-04-11 20:43:11] Error in run 100: Run already exists in server. Run id(s): {8960061}
[2018-04-11 20:43:11] Started run 112 on task 146822, dataset 'segment'.
[2018-04-11 20:43:11] Started run 110 on task 146822, dataset 'segment'.
[2018-04-11 20:43:11] Started run 108 on task 146822, dataset 'segment'.
[2018-04-11 20:43:11] Started run 111 on task 146822, dataset 'segment'.
[2018-04-11 20:43:15] Error in run 105: Run already exists in server. Run id(s): {8960069}
[2018-04-11 20:43:15] Started run 113 on task 146822, dataset 'segment'.
[2018-04-11 20:43:15] Error in run 106: Run already exists in server. Run id(s): {8960070}
[2018-04-11 20:43:15] Started run 114 on task 146822, dataset 'segment'.
[2018-04-11 20:43:15] Error in run 109: Run already exists in server. Run id(s): {8960074}
[2018-04-11 20:43:15] Started run 115 on task 146822, dataset 'segment'.
[2018-04-11 20:43:15] Error in run 108: Run already exists in server. Run id(s): {8960073}
[2018-04-11 20:43:15] Started run 116 on task 146822, dataset 'segment'.
[2018-04-11 20:43:15] Error in run 112: Run already exists in server. Run id(s): {8960081}
[2018-04-11 20:43:15] Error in run 110: Run already exists in server. Run id(s): {8960078}
[2018-04-11 20:43:15] Started run 117 on task 146822, dataset 'segment'.
[2018-04-11 20:43:15] Error in run 111: Run already exists in server. Run id(s): {8960077}
[2018-04-11 20:43:15] Started run 119 on task 146822, dataset 'segment'.
[2018-04-11 20:43:15] Error in run 107: Run already exists in server. Run id(s): {8960071}
[2018-04-11 20:43:15] Started run 118 on task 146822, dataset 'segment'.
[2018-04-11 20:43:15] Started run 120 on task 146822, dataset 'segment'.
[2018-04-11 20:43:19] Error in run 113: Run already exists in server. Run id(s): {8960080}
[2018-04-11 20:43:19] Started run 121 on task 146822, dataset 'segment'.
[2018-04-11 20:43:19] Error in run 114: Run already exists in server. Run id(s): {8960083}
[2018-04-11 20:43:19] Started run 122 on task 146822, dataset 'segment'.
[2018-04-11 20:43:20] Error in run 115: Run already exists in server. Run id(s): {8960085}
[2018-04-11 20:43:20] Started run 123 on task 146822, dataset 'segment'.
[2018-04-11 20:43:20] Error in run 116: Run already exists in server. Run id(s): {8960084}
[2018-04-11 20:43:20] Started run 124 on task 146822, dataset 'segment'.
[2018-04-11 20:43:20] Error in run 118: Run already exists in server. Run id(s): {8960089}
[2018-04-11 20:43:20] Error in run 120: Run already exists in server. Run id(s): {8960092}
[2018-04-11 20:43:20] Started run 125 on task 146822, dataset 'segment'.
[2018-04-11 20:43:20] Error in run 117: Run already exists in server. Run id(s): {8960086}
[2018-04-11 20:43:20] Started run 126 on task 146822, dataset 'segment'.
[2018-04-11 20:43:20] Error in run 119: Run already exists in server. Run id(s): {8960090}
[2018-04-11 20:43:20] Started run 128 on task 146822, dataset 'segment'.
[2018-04-11 20:43:20] Started run 127 on task 146822, dataset 'segment'.
[2018-04-11 20:43:24] Error in run 121: Run already exists in server. Run id(s): {8960093}
[2018-04-11 20:43:24] Error in run 122: Run already exists in server. Run id(s): {8960094}
[2018-04-11 20:43:24] Started run 129 on task 146822, dataset 'segment'.
[2018-04-11 20:43:24] Started run 130 on task 146822, dataset 'segment'.
[2018-04-11 20:43:24] Error in run 123: Run already exists in server. Run id(s): {8960096}
[2018-04-11 20:43:24] Started run 131 on task 146822, dataset 'segment'.
[2018-04-11 20:43:25] Error in run 124: Run already exists in server. Run id(s): {8960097}
[2018-04-11 20:43:25] Started run 132 on task 146822, dataset 'segment'.
[2018-04-11 20:43:25] Error in run 127: Run already exists in server. Run id(s): {8960104}
[2018-04-11 20:43:25] Error in run 126: Run already exists in server. Run id(s): {8960101}
[2018-04-11 20:43:25] Started run 133 on task 146822, dataset 'segment'.
[2018-04-11 20:43:25] Error in run 128: Run already exists in server. Run id(s): {8960103}
[2018-04-11 20:43:25] Started run 135 on task 146822, dataset 'segment'.
[2018-04-11 20:43:25] Error in run 125: Run already exists in server. Run id(s): {8960098}
[2018-04-11 20:43:25] Started run 136 on task 146822, dataset 'segment'.
[2018-04-11 20:43:25] Started run 134 on task 146822, dataset 'segment'.
[2018-04-11 20:43:29] Error in run 129: Run already exists in server. Run id(s): {8960105}
[2018-04-11 20:43:29] Started run 137 on task 146822, dataset 'segment'.
[2018-04-11 20:43:29] Error in run 130: Run already exists in server. Run id(s): {8960106}
[2018-04-11 20:43:29] Started run 138 on task 146822, dataset 'segment'.
[2018-04-11 20:43:29] Error in run 131: Run already exists in server. Run id(s): {8960107}
[2018-04-11 20:43:29] Started run 139 on task 146822, dataset 'segment'.
[2018-04-11 20:43:29] Error in run 132: Run already exists in server. Run id(s): {8960109}
[2018-04-11 20:43:29] Started run 140 on task 146822, dataset 'segment'.
[2018-04-11 20:43:29] Error in run 133: Run already exists in server. Run id(s): {8960110}
[2018-04-11 20:43:29] Started run 141 on task 146822, dataset 'segment'.
[2018-04-11 20:43:30] Error in run 134: Run already exists in server. Run id(s): {8960112}
[2018-04-11 20:43:30] Error in run 135: Run already exists in server. Run id(s): {8960114}
[2018-04-11 20:43:30] Started run 142 on task 146822, dataset 'segment'.
[2018-04-11 20:43:30] Started run 143 on task 146822, dataset 'segment'.
[2018-04-11 20:43:30] Error in run 136: Run already exists in server. Run id(s): {8960116}
[2018-04-11 20:43:30] Started run 144 on task 146822, dataset 'segment'.
[2018-04-11 20:43:33] Error in run 137: Run already exists in server. Run id(s): {8960117}
[2018-04-11 20:43:33] Started run 145 on task 146822, dataset 'segment'.
[2018-04-11 20:43:33] Error in run 139: Run already exists in server. Run id(s): {8960120}
[2018-04-11 20:43:33] Started run 146 on task 146822, dataset 'segment'.
[2018-04-11 20:43:34] Error in run 138: Run already exists in server. Run id(s): {8960119}
[2018-04-11 20:43:34] Started run 147 on task 146822, dataset 'segment'.
[2018-04-11 20:43:34] Error in run 140: Run already exists in server. Run id(s): {8960121}
[2018-04-11 20:43:34] Started run 148 on task 146822, dataset 'segment'.
[2018-04-11 20:43:34] Error in run 141: Run already exists in server. Run id(s): {8960122}
[2018-04-11 20:43:34] Started run 149 on task 146822, dataset 'segment'.
[2018-04-11 20:43:34] Error in run 144: Run already exists in server. Run id(s): {8960126}
[2018-04-11 20:43:34] Error in run 142: Run already exists in server. Run id(s): {8960124}
[2018-04-11 20:43:34] Started run 150 on task 146822, dataset 'segment'.
[2018-04-11 20:43:34] Started run 151 on task 146822, dataset 'segment'.
[2018-04-11 20:43:34] Error in run 143: Run already exists in server. Run id(s): {8960129}
[2018-04-11 20:43:34] Started run 152 on task 146822, dataset 'segment'.
[2018-04-11 20:43:38] Error in run 147: Run already exists in server. Run id(s): {8960132}
[2018-04-11 20:43:38] Error in run 145: Run already exists in server. Run id(s): {8960128}
[2018-04-11 20:43:38] Started run 153 on task 146822, dataset 'segment'.
[2018-04-11 20:43:38] Started run 154 on task 146822, dataset 'segment'.
[2018-04-11 20:43:38] Error in run 146: Run already exists in server. Run id(s): {8960131}
[2018-04-11 20:43:38] Started run 155 on task 146822, dataset 'segment'.
[2018-04-11 20:43:38] Error in run 148: Run already exists in server. Run id(s): {8960133}
[2018-04-11 20:43:38] Started run 156 on task 146822, dataset 'segment'.
[2018-04-11 20:43:38] Error in run 149: Run already exists in server. Run id(s): {8960134}
[2018-04-11 20:43:38] Started run 157 on task 146822, dataset 'segment'.
[2018-04-11 20:43:39] Error in run 152: Run already exists in server. Run id(s): {8960138}
[2018-04-11 20:43:39] Error in run 151: Run already exists in server. Run id(s): {8960139}
[2018-04-11 20:43:39] Started run 158 on task 146822, dataset 'segment'.
[2018-04-11 20:43:39] Error in run 150: Run already exists in server. Run id(s): {8960137}
[2018-04-11 20:43:39] Started run 159 on task 146822, dataset 'segment'.
[2018-04-11 20:43:39] Started run 160 on task 146822, dataset 'segment'.
[2018-04-11 20:43:42] Error in run 154: Run already exists in server. Run id(s): {8960143}
[2018-04-11 20:43:42] Started run 161 on task 146822, dataset 'segment'.
[2018-04-11 20:43:43] Error in run 153: Run already exists in server. Run id(s): {8960140}
[2018-04-11 20:43:43] Started run 162 on task 146822, dataset 'segment'.
[2018-04-11 20:43:43] Error in run 155: Run already exists in server. Run id(s): {8960144}
[2018-04-11 20:43:43] Started run 163 on task 146822, dataset 'segment'.
[2018-04-11 20:43:43] Error in run 157: Run already exists in server. Run id(s): {8960145}
[2018-04-11 20:43:43] Started run 164 on task 146822, dataset 'segment'.
[2018-04-11 20:43:43] Error in run 156: Run already exists in server. Run id(s): {8960146}
[2018-04-11 20:43:43] Started run 165 on task 146822, dataset 'segment'.
[2018-04-11 20:43:44] Error in run 159: Run already exists in server. Run id(s): {8960148}
[2018-04-11 20:43:44] Started run 166 on task 146822, dataset 'segment'.
[2018-04-11 20:43:44] Error in run 158: Run already exists in server. Run id(s): {8960149}
[2018-04-11 20:43:44] Started run 167 on task 146822, dataset 'segment'.
[2018-04-11 20:43:44] Error in run 160: Run already exists in server. Run id(s): {8960151}
[2018-04-11 20:43:44] Started run 168 on task 146822, dataset 'segment'.
[2018-04-11 20:43:47] Error in run 161: Run already exists in server. Run id(s): {8960150}
[2018-04-11 20:43:47] Started run 169 on task 146822, dataset 'segment'.
[2018-04-11 20:43:47] Error in run 162: Run already exists in server. Run id(s): {8960158}
[2018-04-11 20:43:47] Started run 170 on task 146822, dataset 'segment'.
[2018-04-11 20:43:47] Error in run 163: Run already exists in server. Run id(s): {8960155}
[2018-04-11 20:43:47] Started run 171 on task 146822, dataset 'segment'.
[2018-04-11 20:43:48] Error in run 165: Run already exists in server. Run id(s): {8960157}
[2018-04-11 20:43:48] Started run 172 on task 146822, dataset 'segment'.
[2018-04-11 20:43:48] Error in run 164: Run already exists in server. Run id(s): {8960156}
[2018-04-11 20:43:48] Started run 173 on task 146822, dataset 'segment'.
[2018-04-11 20:43:48] Error in run 167: Run already exists in server. Run id(s): {8960162}
[2018-04-11 20:43:48] Error in run 166: Run already exists in server. Run id(s): {8960159}
[2018-04-11 20:43:48] Started run 174 on task 146822, dataset 'segment'.
[2018-04-11 20:43:48] Error in run 168: Run already exists in server. Run id(s): {8960160}
[2018-04-11 20:43:48] Started run 176 on task 146822, dataset 'segment'.
[2018-04-11 20:43:48] Started run 175 on task 146822, dataset 'segment'.
[2018-04-11 20:43:52] Error in run 169: Run already exists in server. Run id(s): {8960164}
[2018-04-11 20:43:52] Started run 177 on task 146822, dataset 'segment'.
[2018-04-11 20:43:52] Error in run 170: Run already exists in server. Run id(s): {8960167}
[2018-04-11 20:43:52] Started run 178 on task 146822, dataset 'segment'.
[2018-04-11 20:43:52] Error in run 171: Run already exists in server. Run id(s): {8960168}
[2018-04-11 20:43:52] Started run 179 on task 146822, dataset 'segment'.
[2018-04-11 20:43:52] Error in run 172: Run already exists in server. Run id(s): {8960169}
[2018-04-11 20:43:52] Started run 180 on task 146822, dataset 'segment'.
[2018-04-11 20:43:52] Error in run 173: Run already exists in server. Run id(s): {8960170}
[2018-04-11 20:43:52] Started run 181 on task 146822, dataset 'segment'.
[2018-04-11 20:43:53] Error in run 174: Run already exists in server. Run id(s): {8960172}
[2018-04-11 20:43:53] Started run 182 on task 146822, dataset 'segment'.
[2018-04-11 20:43:53] Error in run 175: Run already exists in server. Run id(s): {8960171}
[2018-04-11 20:43:53] Started run 183 on task 146822, dataset 'segment'.
[2018-04-11 20:43:53] Error in run 176: Run already exists in server. Run id(s): {8960175}
[2018-04-11 20:43:53] Started run 184 on task 146822, dataset 'segment'.
[2018-04-11 20:43:56] Error in run 177: Run already exists in server. Run id(s): {8960174}
[2018-04-11 20:43:56] Started run 185 on task 146822, dataset 'segment'.
[2018-04-11 20:43:56] Error in run 178: Run already exists in server. Run id(s): {8960179}
[2018-04-11 20:43:56] Started run 186 on task 146822, dataset 'segment'.
[2018-04-11 20:43:56] Error in run 179: Run already exists in server. Run id(s): {8960180}
[2018-04-11 20:43:56] Started run 187 on task 146822, dataset 'segment'.
[2018-04-11 20:43:57] Error in run 180: Run already exists in server. Run id(s): {8960181}
[2018-04-11 20:43:57] Started run 188 on task 146822, dataset 'segment'.
[2018-04-11 20:43:57] Error in run 181: Run already exists in server. Run id(s): {8960182}
[2018-04-11 20:43:57] Started run 189 on task 146822, dataset 'segment'.
[2018-04-11 20:43:58] Error in run 182: Run already exists in server. Run id(s): {8960183}
[2018-04-11 20:43:58] Error in run 183: Run already exists in server. Run id(s): {8960185}
[2018-04-11 20:43:58] Started run 190 on task 146822, dataset 'segment'.
[2018-04-11 20:43:58] Error in run 184: Run already exists in server. Run id(s): {8960184}
[2018-04-11 20:43:58] Started run 192 on task 146822, dataset 'segment'.
[2018-04-11 20:43:58] Started run 191 on task 146822, dataset 'segment'.
[2018-04-11 20:44:01] Error in run 185: Run already exists in server. Run id(s): {8960187}
[2018-04-11 20:44:01] Started run 193 on task 146822, dataset 'segment'.
[2018-04-11 20:44:01] Error in run 187: Run already exists in server. Run id(s): {8960192}
[2018-04-11 20:44:01] Started run 194 on task 146822, dataset 'segment'.
[2018-04-11 20:44:01] Error in run 186: Run already exists in server. Run id(s): {8960191}
[2018-04-11 20:44:01] Started run 195 on task 146822, dataset 'segment'.
[2018-04-11 20:44:01] Error in run 188: Run already exists in server. Run id(s): {8960193}
[2018-04-11 20:44:01] Started run 196 on task 146822, dataset 'segment'.
[2018-04-11 20:44:01] Error in run 189: Run already exists in server. Run id(s): {8960194}
[2018-04-11 20:44:01] Started run 197 on task 146822, dataset 'segment'.
[2018-04-11 20:44:02] Error in run 190: Run already exists in server. Run id(s): {8960196}
[2018-04-11 20:44:02] Error in run 192: Run already exists in server. Run id(s): {8960195}
[2018-04-11 20:44:02] Started run 198 on task 146822, dataset 'segment'.
[2018-04-11 20:44:02] Started run 199 on task 146822, dataset 'segment'.
[2018-04-11 20:44:02] Error in run 191: Run already exists in server. Run id(s): {8960199}
[2018-04-11 20:44:02] Started run 200 on task 146822, dataset 'segment'.
[2018-04-11 20:44:05] Error in run 193: Run already exists in server. Run id(s): {8960197}
[2018-04-11 20:44:05] Started run 201 on task 146822, dataset 'segment'.
[2018-04-11 20:44:06] Error in run 195: Run already exists in server. Run id(s): {8960205}
[2018-04-11 20:44:06] Started run 202 on task 146822, dataset 'segment'.
[2018-04-11 20:44:06] Error in run 194: Run already exists in server. Run id(s): {8960204}
[2018-04-11 20:44:06] Started run 203 on task 146822, dataset 'segment'.
[2018-04-11 20:44:06] Error in run 196: Run already exists in server. Run id(s): {8960203}
[2018-04-11 20:44:06] Started run 204 on task 146822, dataset 'segment'.
[2018-04-11 20:44:06] Error in run 197: Run already exists in server. Run id(s): {8960206}
[2018-04-11 20:44:06] Started run 205 on task 146822, dataset 'segment'.
[2018-04-11 20:44:07] Error in run 200: Run already exists in server. Run id(s): {8960208}
[2018-04-11 20:44:07] Started run 206 on task 146822, dataset 'segment'.
[2018-04-11 20:44:07] Error in run 198: Run already exists in server. Run id(s): {8960207}
[2018-04-11 20:44:07] Started run 207 on task 146822, dataset 'segment'.
[2018-04-11 20:44:07] Error in run 199: Run already exists in server. Run id(s): {8960210}
[2018-04-11 20:44:07] Started run 208 on task 146822, dataset 'segment'.
[2018-04-11 20:44:10] Error in run 201: Run already exists in server. Run id(s): {8960211}
[2018-04-11 20:44:10] Started run 209 on task 146822, dataset 'segment'.
[2018-04-11 20:44:10] Error in run 202: Run already exists in server. Run id(s): {8960215}
[2018-04-11 20:44:10] Started run 210 on task 146822, dataset 'segment'.
[2018-04-11 20:44:10] Error in run 204: Run already exists in server. Run id(s): {8960217}
[2018-04-11 20:44:10] Error in run 205: Run already exists in server. Run id(s): {8960218}
[2018-04-11 20:44:10] Started run 211 on task 146822, dataset 'segment'.
[2018-04-11 20:44:10] Error in run 203: Run already exists in server. Run id(s): {8960216}
[2018-04-11 20:44:10] Started run 213 on task 146822, dataset 'segment'.
[2018-04-11 20:44:10] Started run 212 on task 146822, dataset 'segment'.
[2018-04-11 20:44:12] Error in run 207: Run already exists in server. Run id(s): {8960219}
[2018-04-11 20:44:12] Started run 214 on task 146822, dataset 'segment'.
[2018-04-11 20:44:12] Error in run 206: Run already exists in server. Run id(s): {8960220}
[2018-04-11 20:44:12] Started run 215 on task 146822, dataset 'segment'.
[2018-04-11 20:44:12] Error in run 208: Run already exists in server. Run id(s): {8960221}
[2018-04-11 20:44:12] Started run 216 on task 146822, dataset 'segment'.
[2018-04-11 20:44:14] Error in run 209: Run already exists in server. Run id(s): {8960223}
[2018-04-11 20:44:14] Started run 217 on task 146822, dataset 'segment'.
[2018-04-11 20:44:15] Error in run 210: Run already exists in server. Run id(s): {8960228}
[2018-04-11 20:44:15] Started run 218 on task 146822, dataset 'segment'.
[2018-04-11 20:44:15] Error in run 213: Run already exists in server. Run id(s): {8960230}
[2018-04-11 20:44:15] Error in run 212: Run already exists in server. Run id(s): {8960229}
[2018-04-11 20:44:15] Started run 219 on task 146822, dataset 'segment'.
[2018-04-11 20:44:15] Started run 220 on task 146822, dataset 'segment'.
[2018-04-11 20:44:15] Error in run 211: Run already exists in server. Run id(s): {8960227}
[2018-04-11 20:44:15] Started run 221 on task 146822, dataset 'segment'.
[2018-04-11 20:44:16] Error in run 215: Run already exists in server. Run id(s): {8960232}
[2018-04-11 20:44:16] Started run 222 on task 146822, dataset 'segment'.
[2018-04-11 20:44:16] Error in run 214: Run already exists in server. Run id(s): {8960231}
[2018-04-11 20:44:16] Started run 223 on task 146822, dataset 'segment'.
[2018-04-11 20:44:16] Error in run 216: Run already exists in server. Run id(s): {8960234}
[2018-04-11 20:44:16] Started run 224 on task 146822, dataset 'segment'.
[2018-04-11 20:44:19] Error in run 217: Run already exists in server. Run id(s): {8960236}
[2018-04-11 20:44:19] Started run 225 on task 146822, dataset 'segment'.
[2018-04-11 20:44:19] Error in run 218: Run already exists in server. Run id(s): {8960240}
[2018-04-11 20:44:19] Started run 226 on task 146822, dataset 'segment'.
[2018-04-11 20:44:19] Error in run 219: Run already exists in server. Run id(s): {8960239}
[2018-04-11 20:44:19] Started run 227 on task 146822, dataset 'segment'.
[2018-04-11 20:44:20] Error in run 221: Run already exists in server. Run id(s): {8960242}
[2018-04-11 20:44:20] Error in run 220: Run already exists in server. Run id(s): {8960241}
[2018-04-11 20:44:20] Started run 228 on task 146822, dataset 'segment'.
[2018-04-11 20:44:20] Started run 229 on task 146822, dataset 'segment'.
[2018-04-11 20:44:21] Error in run 222: Run already exists in server. Run id(s): {8960243}
[2018-04-11 20:44:21] Started run 230 on task 146822, dataset 'segment'.
[2018-04-11 20:44:21] Error in run 223: Run already exists in server. Run id(s): {8960244}
[2018-04-11 20:44:21] Started run 231 on task 146822, dataset 'segment'.
[2018-04-11 20:44:21] Error in run 224: Run already exists in server. Run id(s): {8960245}
[2018-04-11 20:44:21] Started run 232 on task 146822, dataset 'segment'.
[2018-04-11 20:44:23] Error in run 225: Run already exists in server. Run id(s): {8960247}
[2018-04-11 20:44:23] Started run 233 on task 146822, dataset 'segment'.
[2018-04-11 20:44:24] Error in run 226: Run already exists in server. Run id(s): {8960250}
[2018-04-11 20:44:24] Started run 234 on task 146822, dataset 'segment'.
[2018-04-11 20:44:24] Error in run 227: Run already exists in server. Run id(s): {8960251}
[2018-04-11 20:44:24] Started run 235 on task 146822, dataset 'segment'.
[2018-04-11 20:44:24] Error in run 229: Run already exists in server. Run id(s): {8960254}
[2018-04-11 20:44:24] Started run 236 on task 146822, dataset 'segment'.
[2018-04-11 20:44:24] Error in run 228: Run already exists in server. Run id(s): {8960253}
[2018-04-11 20:44:24] Started run 237 on task 146822, dataset 'segment'.
[2018-04-11 20:44:25] Error in run 230: Run already exists in server. Run id(s): {8960255}
[2018-04-11 20:44:25] Started run 238 on task 146822, dataset 'segment'.
[2018-04-11 20:44:25] Error in run 231: Run already exists in server. Run id(s): {8960256}
[2018-04-11 20:44:25] Started run 239 on task 146822, dataset 'segment'.
[2018-04-11 20:44:25] Error in run 232: Run already exists in server. Run id(s): {8960257}
[2018-04-11 20:44:25] Started run 240 on task 146822, dataset 'segment'.
[2018-04-11 20:44:28] Error in run 233: Run already exists in server. Run id(s): {8960259}
[2018-04-11 20:44:28] Started run 241 on task 146822, dataset 'segment'.
[2018-04-11 20:44:28] Error in run 234: Run already exists in server. Run id(s): {8960261}
[2018-04-11 20:44:28] Started run 242 on task 146822, dataset 'segment'.
[2018-04-11 20:44:28] Error in run 235: Run already exists in server. Run id(s): {8960262}
[2018-04-11 20:44:28] Started run 243 on task 146822, dataset 'segment'.
[2018-04-11 20:44:29] Error in run 237: Run already exists in server. Run id(s): {8960266}
[2018-04-11 20:44:29] Started run 244 on task 146822, dataset 'segment'.
[2018-04-11 20:44:29] Error in run 236: Run already exists in server. Run id(s): {8960264}
[2018-04-11 20:44:29] Started run 245 on task 146822, dataset 'segment'.
[2018-04-11 20:44:30] Error in run 239: Run already exists in server. Run id(s): {8960267}
[2018-04-11 20:44:30] Error in run 240: Run already exists in server. Run id(s): {8960269}
[2018-04-11 20:44:30] Started run 246 on task 146822, dataset 'segment'.
[2018-04-11 20:44:30] Error in run 238: Run already exists in server. Run id(s): {8960268}
[2018-04-11 20:44:30] Started run 248 on task 146822, dataset 'segment'.
[2018-04-11 20:44:30] Started run 247 on task 146822, dataset 'segment'.
[2018-04-11 20:44:32] Error in run 241: Run already exists in server. Run id(s): {8960272}
[2018-04-11 20:44:32] Started run 249 on task 146822, dataset 'segment'.
[2018-04-11 20:44:33] Error in run 243: Run already exists in server. Run id(s): {8960273}
[2018-04-11 20:44:33] Started run 250 on task 146822, dataset 'segment'.
[2018-04-11 20:44:33] Error in run 242: Run already exists in server. Run id(s): {8960274}
[2018-04-11 20:44:33] Started run 251 on task 146822, dataset 'segment'.
[2018-04-11 20:44:33] Error in run 244: Run already exists in server. Run id(s): {8960275}
[2018-04-11 20:44:33] Error in run 245: Run already exists in server. Run id(s): {8960278}
[2018-04-11 20:44:33] Started run 252 on task 146822, dataset 'segment'.
[2018-04-11 20:44:33] Started run 253 on task 146822, dataset 'segment'.
[2018-04-11 20:44:34] Error in run 248: Run already exists in server. Run id(s): {8960281}
[2018-04-11 20:44:34] Started run 254 on task 146822, dataset 'segment'.
[2018-04-11 20:44:34] Error in run 247: Run already exists in server. Run id(s): {8960280}
[2018-04-11 20:44:34] Started run 255 on task 146822, dataset 'segment'.
[2018-04-11 20:44:34] Error in run 246: Run already exists in server. Run id(s): {8960279}
[2018-04-11 20:44:34] Started run 256 on task 146822, dataset 'segment'.
[2018-04-11 20:44:37] Error in run 249: Run already exists in server. Run id(s): {8960284}
[2018-04-11 20:44:37] Started run 257 on task 146822, dataset 'segment'.
[2018-04-11 20:44:37] Error in run 250: Run already exists in server. Run id(s): {8960286}
[2018-04-11 20:44:37] Started run 258 on task 146822, dataset 'segment'.
[2018-04-11 20:44:37] Error in run 251: Run already exists in server. Run id(s): {8960285}
[2018-04-11 20:44:37] Started run 259 on task 146822, dataset 'segment'.
[2018-04-11 20:44:38] Error in run 252: Run already exists in server. Run id(s): {8960287}
[2018-04-11 20:44:38] Started run 260 on task 146822, dataset 'segment'.
[2018-04-11 20:44:38] Error in run 253: Run already exists in server. Run id(s): {8960290}
[2018-04-11 20:44:38] Started run 261 on task 146822, dataset 'segment'.
[2018-04-11 20:44:39] Error in run 254: Run already exists in server. Run id(s): {8960293}
[2018-04-11 20:44:39] Started run 262 on task 146822, dataset 'segment'.
[2018-04-11 20:44:39] Error in run 255: Run already exists in server. Run id(s): {8960292}
[2018-04-11 20:44:39] Started run 263 on task 146822, dataset 'segment'.
[2018-04-11 20:44:39] Error in run 256: Run already exists in server. Run id(s): {8960291}
[2018-04-11 20:44:39] Started run 264 on task 146822, dataset 'segment'.
[2018-04-11 20:44:41] Error in run 257: Run already exists in server. Run id(s): {8960295}
[2018-04-11 20:44:41] Started run 265 on task 146822, dataset 'segment'.
[2018-04-11 20:44:42] Error in run 258: Run already exists in server. Run id(s): {8960298}
[2018-04-11 20:44:42] Started run 266 on task 146822, dataset 'segment'.
[2018-04-11 20:44:42] Error in run 259: Run already exists in server. Run id(s): {8960299}
[2018-04-11 20:44:42] Started run 267 on task 146822, dataset 'segment'.
[2018-04-11 20:44:42] Error in run 260: Run already exists in server. Run id(s): {8960300}
[2018-04-11 20:44:42] Started run 268 on task 146822, dataset 'segment'.
[2018-04-11 20:44:42] Error in run 261: Run already exists in server. Run id(s): {8960302}
[2018-04-11 20:44:42] Started run 269 on task 146822, dataset 'segment'.
[2018-04-11 20:44:43] Error in run 262: Run already exists in server. Run id(s): {8960303}
[2018-04-11 20:44:43] Started run 270 on task 146822, dataset 'segment'.
[2018-04-11 20:44:43] Error in run 263: Run already exists in server. Run id(s): {8960304}
[2018-04-11 20:44:43] Started run 271 on task 146822, dataset 'segment'.
[2018-04-11 20:44:44] Error in run 264: Run already exists in server. Run id(s): {8960305}
[2018-04-11 20:44:44] Started run 272 on task 146822, dataset 'segment'.
[2018-04-11 20:44:46] Error in run 265: Run already exists in server. Run id(s): {8960307}
[2018-04-11 20:44:46] Started run 273 on task 146822, dataset 'segment'.
[2018-04-11 20:44:46] Error in run 266: Run already exists in server. Run id(s): {8960309}
[2018-04-11 20:44:46] Started run 274 on task 146822, dataset 'segment'.
[2018-04-11 20:44:46] Error in run 267: Run already exists in server. Run id(s): {8960312}
[2018-04-11 20:44:46] Started run 275 on task 146822, dataset 'segment'.
[2018-04-11 20:44:47] Error in run 268: Run already exists in server. Run id(s): {8960311}
[2018-04-11 20:44:47] Started run 276 on task 146822, dataset 'segment'.
[2018-04-11 20:44:47] Error in run 269: Run already exists in server. Run id(s): {8960314}
[2018-04-11 20:44:47] Started run 277 on task 146822, dataset 'segment'.
[2018-04-11 20:44:48] Error in run 271: Run already exists in server. Run id(s): {8960317}
[2018-04-11 20:44:48] Error in run 270: Run already exists in server. Run id(s): {8960315}
[2018-04-11 20:44:48] Started run 278 on task 146822, dataset 'segment'.
[2018-04-11 20:44:48] Started run 279 on task 146822, dataset 'segment'.
[2018-04-11 20:44:48] Error in run 272: Run already exists in server. Run id(s): {8960316}
[2018-04-11 20:44:48] Started run 280 on task 146822, dataset 'segment'.
[2018-04-11 20:44:50] Error in run 273: Run already exists in server. Run id(s): {8960319}
[2018-04-11 20:44:50] Started run 281 on task 146822, dataset 'segment'.
[2018-04-11 20:44:51] Error in run 274: Run already exists in server. Run id(s): {8960321}
[2018-04-11 20:44:51] Started run 282 on task 146822, dataset 'segment'.
[2018-04-11 20:44:51] Error in run 275: Run already exists in server. Run id(s): {8960323}
[2018-04-11 20:44:51] Started run 283 on task 146822, dataset 'segment'.
[2018-04-11 20:44:51] Error in run 276: Run already exists in server. Run id(s): {8960325}
[2018-04-11 20:44:51] Started run 284 on task 146822, dataset 'segment'.
[2018-04-11 20:44:51] Error in run 277: Run already exists in server. Run id(s): {8960326}
[2018-04-11 20:44:51] Started run 285 on task 146822, dataset 'segment'.
[2018-04-11 20:44:53] Error in run 279: Run already exists in server. Run id(s): {8960328}
[2018-04-11 20:44:53] Started run 286 on task 146822, dataset 'segment'.
[2018-04-11 20:44:53] Error in run 278: Run already exists in server. Run id(s): {8960327}
[2018-04-11 20:44:53] Started run 287 on task 146822, dataset 'segment'.
[2018-04-11 20:44:53] Error in run 280: Run already exists in server. Run id(s): {8960329}
[2018-04-11 20:44:53] Started run 288 on task 146822, dataset 'segment'.
[2018-04-11 20:44:55] Error in run 281: Run already exists in server. Run id(s): {8960331}
[2018-04-11 20:44:55] Started run 289 on task 146822, dataset 'segment'.
[2018-04-11 20:44:55] Error in run 282: Run already exists in server. Run id(s): {8960332}
[2018-04-11 20:44:55] Started run 290 on task 146822, dataset 'segment'.
[2018-04-11 20:44:55] Error in run 283: Run already exists in server. Run id(s): {8960334}
[2018-04-11 20:44:55] Started run 291 on task 146822, dataset 'segment'.
[2018-04-11 20:44:56] Error in run 284: Run already exists in server. Run id(s): {8960336}
[2018-04-11 20:44:56] Started run 292 on task 146822, dataset 'segment'.
[2018-04-11 20:44:56] Error in run 285: Run already exists in server. Run id(s): {8960339}
[2018-04-11 20:44:56] Started run 293 on task 146822, dataset 'segment'.
[2018-04-11 20:44:57] Error in run 287: Run already exists in server. Run id(s): {8960340}
[2018-04-11 20:44:57] Started run 294 on task 146822, dataset 'segment'.
[2018-04-11 20:44:57] Error in run 286: Run already exists in server. Run id(s): {8960338}
[2018-04-11 20:44:57] Started run 295 on task 146822, dataset 'segment'.
[2018-04-11 20:44:57] Error in run 288: Run already exists in server. Run id(s): {8960341}
[2018-04-11 20:44:57] Started run 296 on task 146822, dataset 'segment'.
[2018-04-11 20:45:00] Error in run 289: Run already exists in server. Run id(s): {8960345}
[2018-04-11 20:45:00] Started run 297 on task 146822, dataset 'segment'.
[2018-04-11 20:45:00] Error in run 290: Run already exists in server. Run id(s): {8960343}
[2018-04-11 20:45:00] Started run 298 on task 146822, dataset 'segment'.
[2018-04-11 20:45:00] Error in run 291: Run already exists in server. Run id(s): {8960346}
[2018-04-11 20:45:00] Started run 299 on task 146822, dataset 'segment'.
[2018-04-11 20:45:00] Error in run 292: Run already exists in server. Run id(s): {8960348}
[2018-04-11 20:45:00] Started run 300 on task 146822, dataset 'segment'.
[2018-04-11 20:45:00] Error in run 293: Run already exists in server. Run id(s): {8960350}
[2018-04-11 20:45:00] Started run 301 on task 146822, dataset 'segment'.
[2018-04-11 20:45:01] Error in run 294: Run already exists in server. Run id(s): {8960352}
[2018-04-11 20:45:01] Started run 302 on task 146822, dataset 'segment'.
[2018-04-11 20:45:01] Error in run 295: Run already exists in server. Run id(s): {8960351}
[2018-04-11 20:45:01] Started run 303 on task 146822, dataset 'segment'.
[2018-04-11 20:45:02] Error in run 296: Run already exists in server. Run id(s): {8960354}
[2018-04-11 20:45:02] Started run 304 on task 146822, dataset 'segment'.
[2018-04-11 20:45:04] Error in run 297: Run already exists in server. Run id(s): {8960355}
[2018-04-11 20:45:04] Started run 305 on task 146822, dataset 'segment'.
[2018-04-11 20:45:04] Error in run 298: Run already exists in server. Run id(s): {8960356}
[2018-04-11 20:45:04] Started run 306 on task 146822, dataset 'segment'.
[2018-04-11 20:45:04] Error in run 299: Run already exists in server. Run id(s): {8960359}
[2018-04-11 20:45:04] Started run 307 on task 146822, dataset 'segment'.
[2018-04-11 20:45:05] Error in run 300: Run already exists in server. Run id(s): {8960358}
[2018-04-11 20:45:05] Started run 308 on task 146822, dataset 'segment'.
[2018-04-11 20:45:05] Error in run 301: Run already exists in server. Run id(s): {8960362}
[2018-04-11 20:45:05] Started run 309 on task 146822, dataset 'segment'.
[2018-04-11 20:45:06] Error in run 302: Run already exists in server. Run id(s): {8960364}
[2018-04-11 20:45:06] Started run 310 on task 146822, dataset 'segment'.
[2018-04-11 20:45:06] Error in run 303: Run already exists in server. Run id(s): {8960363}
[2018-04-11 20:45:06] Started run 311 on task 146822, dataset 'segment'.
[2018-04-11 20:45:06] Error in run 304: Run already exists in server. Run id(s): {8960365}
[2018-04-11 20:45:06] Started run 312 on task 146822, dataset 'segment'.
[2018-04-11 20:45:09] Error in run 305: Run already exists in server. Run id(s): {8960367}
[2018-04-11 20:45:09] Started run 313 on task 146822, dataset 'segment'.
[2018-04-11 20:45:09] Error in run 306: Run already exists in server. Run id(s): {8960368}
[2018-04-11 20:45:09] Started run 314 on task 146822, dataset 'segment'.
[2018-04-11 20:45:09] Error in run 307: Run already exists in server. Run id(s): {8960370}
[2018-04-11 20:45:09] Started run 315 on task 146822, dataset 'segment'.
[2018-04-11 20:45:10] Error in run 308: Run already exists in server. Run id(s): {8960372}
[2018-04-11 20:45:10] Started run 316 on task 146822, dataset 'segment'.
[2018-04-11 20:45:10] Error in run 309: Run already exists in server. Run id(s): {8960374}
[2018-04-11 20:45:10] Started run 317 on task 146822, dataset 'segment'.
[2018-04-11 20:45:11] Error in run 311: Run already exists in server. Run id(s): {8960376}
[2018-04-11 20:45:11] Started run 318 on task 146822, dataset 'segment'.
[2018-04-11 20:45:11] Error in run 310: Run already exists in server. Run id(s): {8960375}
[2018-04-11 20:45:11] Started run 319 on task 146822, dataset 'segment'.
[2018-04-11 20:45:11] Error in run 312: Run already exists in server. Run id(s): {8960378}
[2018-04-11 20:45:11] Started run 320 on task 146822, dataset 'segment'.
[2018-04-11 20:45:13] Error in run 313: Run already exists in server. Run id(s): {8960379}
[2018-04-11 20:45:13] Started run 321 on task 146822, dataset 'segment'.
[2018-04-11 20:45:13] Error in run 315: Run already exists in server. Run id(s): {8960383}
[2018-04-11 20:45:13] Started run 322 on task 146822, dataset 'segment'.
[2018-04-11 20:45:14] Error in run 314: Run already exists in server. Run id(s): {8960380}
[2018-04-11 20:45:14] Started run 323 on task 146822, dataset 'segment'.
[2018-04-11 20:45:14] Error in run 316: Run already exists in server. Run id(s): {8960385}
[2018-04-11 20:45:14] Started run 324 on task 146822, dataset 'segment'.
[2018-04-11 20:45:14] Error in run 317: Run already exists in server. Run id(s): {8960387}
[2018-04-11 20:45:14] Started run 325 on task 146822, dataset 'segment'.
[2018-04-11 20:45:15] Error in run 319: Run already exists in server. Run id(s): {8960388}
[2018-04-11 20:45:15] Started run 326 on task 146822, dataset 'segment'.
[2018-04-11 20:45:15] Error in run 318: Run already exists in server. Run id(s): {8960386}
[2018-04-11 20:45:15] Started run 327 on task 146822, dataset 'segment'.
[2018-04-11 20:45:16] Error in run 320: Run already exists in server. Run id(s): {8960390}
[2018-04-11 20:45:16] Started run 328 on task 146822, dataset 'segment'.
[2018-04-11 20:45:18] Error in run 321: Run already exists in server. Run id(s): {8960391}
[2018-04-11 20:45:18] Started run 329 on task 146822, dataset 'segment'.
[2018-04-11 20:45:18] Error in run 322: Run already exists in server. Run id(s): {8960392}
[2018-04-11 20:45:18] Started run 330 on task 146822, dataset 'segment'.
[2018-04-11 20:45:18] Error in run 323: Run already exists in server. Run id(s): {8960394}
[2018-04-11 20:45:18] Started run 331 on task 146822, dataset 'segment'.
[2018-04-11 20:45:19] Error in run 324: Run already exists in server. Run id(s): {8960397}
[2018-04-11 20:45:19] Started run 332 on task 146822, dataset 'segment'.
[2018-04-11 20:45:19] Error in run 325: Run already exists in server. Run id(s): {8960398}
[2018-04-11 20:45:19] Started run 333 on task 146822, dataset 'segment'.
[2018-04-11 20:45:20] Error in run 326: Run already exists in server. Run id(s): {8960399}
[2018-04-11 20:45:20] Started run 334 on task 146822, dataset 'segment'.
[2018-04-11 20:45:20] Error in run 327: Run already exists in server. Run id(s): {8960400}
[2018-04-11 20:45:20] Started run 335 on task 146822, dataset 'segment'.
[2018-04-11 20:45:20] Error in run 328: Run already exists in server. Run id(s): {8960402}
[2018-04-11 20:45:20] Started run 336 on task 146822, dataset 'segment'.
[2018-04-11 20:45:22] Error in run 329: Run already exists in server. Run id(s): {8960403}
[2018-04-11 20:45:22] Started run 337 on task 146822, dataset 'segment'.
[2018-04-11 20:45:23] Error in run 330: Run already exists in server. Run id(s): {8960405}
[2018-04-11 20:45:23] Started run 338 on task 146822, dataset 'segment'.
[2018-04-11 20:45:23] Error in run 331: Run already exists in server. Run id(s): {8960404}
[2018-04-11 20:45:23] Started run 339 on task 146822, dataset 'segment'.
[2018-04-11 20:45:23] Error in run 332: Run already exists in server. Run id(s): {8960408}
[2018-04-11 20:45:23] Started run 340 on task 146822, dataset 'segment'.
[2018-04-11 20:45:24] Error in run 333: Run already exists in server. Run id(s): {8960410}
[2018-04-11 20:45:24] Started run 341 on task 146822, dataset 'segment'.
[2018-04-11 20:45:24] Error in run 334: Run already exists in server. Run id(s): {8960411}
[2018-04-11 20:45:24] Started run 342 on task 146822, dataset 'segment'.
[2018-04-11 20:45:24] Error in run 335: Run already exists in server. Run id(s): {8960412}
[2018-04-11 20:45:24] Started run 343 on task 146822, dataset 'segment'.
[2018-04-11 20:45:25] Error in run 336: Run already exists in server. Run id(s): {8960414}
[2018-04-11 20:45:25] Started run 344 on task 146822, dataset 'segment'.
[2018-04-11 20:45:27] Error in run 337: Run already exists in server. Run id(s): {8960415}
[2018-04-11 20:45:27] Started run 345 on task 146822, dataset 'segment'.
[2018-04-11 20:45:27] Error in run 338: Run already exists in server. Run id(s): {8960416}
[2018-04-11 20:45:27] Started run 346 on task 146822, dataset 'segment'.
[2018-04-11 20:45:27] Error in run 339: Run already exists in server. Run id(s): {8960417}
[2018-04-11 20:45:27] Started run 347 on task 146822, dataset 'segment'.
[2018-04-11 20:45:28] Error in run 340: Run already exists in server. Run id(s): {8960420}
[2018-04-11 20:45:28] Started run 348 on task 146822, dataset 'segment'.
[2018-04-11 20:45:28] Error in run 341: Run already exists in server. Run id(s): {8960421}
[2018-04-11 20:45:28] Started run 349 on task 146822, dataset 'segment'.
[2018-04-11 20:45:29] Error in run 342: Run already exists in server. Run id(s): {8960423}
[2018-04-11 20:45:29] Started run 350 on task 146822, dataset 'segment'.
[2018-04-11 20:45:29] Error in run 343: Run already exists in server. Run id(s): {8960424}
[2018-04-11 20:45:29] Started run 351 on task 146822, dataset 'segment'.
[2018-04-11 20:45:29] Error in run 344: Run already exists in server. Run id(s): {8960427}
[2018-04-11 20:45:29] Started run 352 on task 146822, dataset 'segment'.
[2018-04-11 20:45:32] Error in run 345: Run already exists in server. Run id(s): {8960426}
[2018-04-11 20:45:32] Started run 353 on task 146822, dataset 'segment'.
[2018-04-11 20:45:32] Error in run 346: Run already exists in server. Run id(s): {8960428}
[2018-04-11 20:45:32] Started run 354 on task 146822, dataset 'segment'.
[2018-04-11 20:45:32] Error in run 347: Run already exists in server. Run id(s): {8960430}
[2018-04-11 20:45:32] Started run 355 on task 146822, dataset 'segment'.
[2018-04-11 20:45:32] Error in run 348: Run already exists in server. Run id(s): {8960431}
[2018-04-11 20:45:32] Started run 356 on task 146822, dataset 'segment'.
[2018-04-11 20:45:33] Error in run 349: Run already exists in server. Run id(s): {8960434}
[2018-04-11 20:45:33] Started run 357 on task 146822, dataset 'segment'.
[2018-04-11 20:45:33] Error in run 350: Run already exists in server. Run id(s): {8960435}
[2018-04-11 20:45:33] Started run 358 on task 146822, dataset 'segment'.
[2018-04-11 20:45:33] Error in run 351: Run already exists in server. Run id(s): {8960436}
[2018-04-11 20:45:33] Started run 359 on task 146822, dataset 'segment'.
[2018-04-11 20:45:34] Error in run 352: Run already exists in server. Run id(s): {8960437}
[2018-04-11 20:45:34] Started run 360 on task 146822, dataset 'segment'.
[2018-04-11 20:45:36] Error in run 354: Run already exists in server. Run id(s): {8960440, 9014854}
[2018-04-11 20:45:36] Started run 361 on task 146822, dataset 'segment'.
[2018-04-11 20:45:36] Error in run 353: Run already exists in server. Run id(s): {8960439}
[2018-04-11 20:45:36] Started run 362 on task 146822, dataset 'segment'.
[2018-04-11 20:45:36] Error in run 355: Run already exists in server. Run id(s): {8960443}
[2018-04-11 20:45:36] Started run 363 on task 146822, dataset 'segment'.
[2018-04-11 20:45:37] Error in run 356: Run already exists in server. Run id(s): {8960442}
[2018-04-11 20:45:37] Started run 364 on task 146822, dataset 'segment'.
[2018-04-11 20:45:37] Error in run 357: Run already exists in server. Run id(s): {8960444}
[2018-04-11 20:45:37] Started run 365 on task 146822, dataset 'segment'.
[2018-04-11 20:45:38] Error in run 358: Run already exists in server. Run id(s): {8960447}
[2018-04-11 20:45:38] Started run 366 on task 146822, dataset 'segment'.
[2018-04-11 20:45:38] Error in run 359: Run already exists in server. Run id(s): {8960448}
[2018-04-11 20:45:38] Started run 367 on task 146822, dataset 'segment'.
[2018-04-11 20:45:38] Error in run 360: Run already exists in server. Run id(s): {8960450}
[2018-04-11 20:45:38] Started run 368 on task 146822, dataset 'segment'.
[2018-04-11 20:45:41] Error in run 361: Run already exists in server. Run id(s): {8960451}
[2018-04-11 20:45:41] Started run 369 on task 146822, dataset 'segment'.
[2018-04-11 20:45:41] Error in run 363: Run already exists in server. Run id(s): {8960454}
[2018-04-11 20:45:41] Error in run 362: Run already exists in server. Run id(s): {8960452}
[2018-04-11 20:45:41] Started run 370 on task 146822, dataset 'segment'.
[2018-04-11 20:45:41] Started run 371 on task 146822, dataset 'segment'.
[2018-04-11 20:45:42] Error in run 364: Run already exists in server. Run id(s): {8960455}
[2018-04-11 20:45:42] Started run 372 on task 146822, dataset 'segment'.
[2018-04-11 20:45:42] Error in run 365: Run already exists in server. Run id(s): {8960456}
[2018-04-11 20:45:42] Started run 373 on task 146822, dataset 'segment'.
[2018-04-11 20:45:43] Error in run 366: Run already exists in server. Run id(s): {8960459}
[2018-04-11 20:45:43] Error in run 367: Run already exists in server. Run id(s): {8960461}
[2018-04-11 20:45:43] Started run 374 on task 146822, dataset 'segment'.
[2018-04-11 20:45:43] Started run 375 on task 146822, dataset 'segment'.
[2018-04-11 20:45:43] Error in run 368: Run already exists in server. Run id(s): {8960460}
[2018-04-11 20:45:43] Started run 376 on task 146822, dataset 'segment'.
[2018-04-11 20:45:46] Error in run 370: Run already exists in server. Run id(s): {8960464}
[2018-04-11 20:45:46] Error in run 369: Run already exists in server. Run id(s): {8960463}
[2018-04-11 20:45:46] Started run 377 on task 146822, dataset 'segment'.
[2018-04-11 20:45:46] Started run 378 on task 146822, dataset 'segment'.
[2018-04-11 20:45:46] Error in run 371: Run already exists in server. Run id(s): {8960467}
[2018-04-11 20:45:46] Started run 379 on task 146822, dataset 'segment'.
[2018-04-11 20:45:46] Error in run 373: Run already exists in server. Run id(s): {8960469}
[2018-04-11 20:45:46] Started run 380 on task 146822, dataset 'segment'.
[2018-04-11 20:45:46] Error in run 372: Run already exists in server. Run id(s): {8960466}
[2018-04-11 20:45:46] Started run 381 on task 146822, dataset 'segment'.
[2018-04-11 20:45:47] Error in run 375: Run already exists in server. Run id(s): {8960472}
[2018-04-11 20:45:47] Error in run 376: Run already exists in server. Run id(s): {8960473}
[2018-04-11 20:45:47] Started run 382 on task 146822, dataset 'segment'.
[2018-04-11 20:45:47] Error in run 374: Run already exists in server. Run id(s): {8960471}
[2018-04-11 20:45:47] Started run 384 on task 146822, dataset 'segment'.
[2018-04-11 20:45:47] Started run 383 on task 146822, dataset 'segment'.
[2018-04-11 20:45:50] Error in run 377: Run already exists in server. Run id(s): {8960475}
[2018-04-11 20:45:50] Started run 385 on task 146822, dataset 'segment'.
[2018-04-11 20:45:50] Error in run 379: Run already exists in server. Run id(s): {8960477}
[2018-04-11 20:45:50] Error in run 378: Run already exists in server. Run id(s): {8960476}
[2018-04-11 20:45:50] Started run 386 on task 146822, dataset 'segment'.
[2018-04-11 20:45:50] Started run 387 on task 146822, dataset 'segment'.
[2018-04-11 20:45:51] Error in run 380: Run already exists in server. Run id(s): {8960479}
[2018-04-11 20:45:51] Started run 388 on task 146822, dataset 'segment'.
[2018-04-11 20:45:51] Error in run 381: Run already exists in server. Run id(s): {8960480}
[2018-04-11 20:45:51] Started run 389 on task 146822, dataset 'segment'.
[2018-04-11 20:45:52] Error in run 383: Run already exists in server. Run id(s): {8960484}
[2018-04-11 20:45:52] Started run 390 on task 146822, dataset 'segment'.
[2018-04-11 20:45:52] Error in run 384: Run already exists in server. Run id(s): {8960485}
[2018-04-11 20:45:52] Error in run 382: Run already exists in server. Run id(s): {8960483}
[2018-04-11 20:45:52] Started run 391 on task 146822, dataset 'segment'.
[2018-04-11 20:45:52] Started run 392 on task 146822, dataset 'segment'.
[2018-04-11 20:45:55] Error in run 385: Run already exists in server. Run id(s): {8960486}
[2018-04-11 20:45:55] Error in run 387: Run already exists in server. Run id(s): {8960489}
[2018-04-11 20:45:55] Error in run 386: Run already exists in server. Run id(s): {8960488}
[2018-04-11 20:45:55] Started run 393 on task 146822, dataset 'segment'.
[2018-04-11 20:45:55] Started run 395 on task 146822, dataset 'segment'.
[2018-04-11 20:45:55] Started run 394 on task 146822, dataset 'segment'.
[2018-04-11 20:45:56] Error in run 389: Run already exists in server. Run id(s): {8960492}
[2018-04-11 20:45:56] Error in run 388: Run already exists in server. Run id(s): {8960491}
[2018-04-11 20:45:56] Started run 396 on task 146822, dataset 'segment'.
[2018-04-11 20:45:56] Started run 397 on task 146822, dataset 'segment'.
[2018-04-11 20:45:57] Error in run 392: Run already exists in server. Run id(s): {8960497}
[2018-04-11 20:45:57] Error in run 391: Run already exists in server. Run id(s): {8960496}
[2018-04-11 20:45:57] Error in run 390: Run already exists in server. Run id(s): {8960494}
[2018-04-11 20:45:57] Started run 398 on task 146822, dataset 'segment'.
[2018-04-11 20:45:57] Started run 399 on task 146822, dataset 'segment'.
[2018-04-11 20:45:57] Started run 400 on task 146822, dataset 'segment'.
[2018-04-11 20:46:00] Error in run 395: Run already exists in server. Run id(s): {8960501}
[2018-04-11 20:46:00] Started run 401 on task 146822, dataset 'segment'.
[2018-04-11 20:46:00] Error in run 394: Run already exists in server. Run id(s): {8960500}
[2018-04-11 20:46:00] Started run 402 on task 146822, dataset 'segment'.
[2018-04-11 20:46:00] Error in run 393: Run already exists in server. Run id(s): {8960498}
[2018-04-11 20:46:00] Started run 403 on task 146822, dataset 'segment'.
[2018-04-11 20:46:01] Error in run 396: Run already exists in server. Run id(s): {8960503}
[2018-04-11 20:46:01] Error in run 397: Run already exists in server. Run id(s): {8960504}
[2018-04-11 20:46:01] Started run 404 on task 146822, dataset 'segment'.
[2018-04-11 20:46:01] Started run 405 on task 146822, dataset 'segment'.
[2018-04-11 20:46:01] Error in run 399: Run already exists in server. Run id(s): {8960508}
[2018-04-11 20:46:01] Started run 406 on task 146822, dataset 'segment'.
[2018-04-11 20:46:01] Error in run 398: Run already exists in server. Run id(s): {8960505}
[2018-04-11 20:46:01] Started run 407 on task 146822, dataset 'segment'.
[2018-04-11 20:46:01] Error in run 400: Run already exists in server. Run id(s): {8960509}
[2018-04-11 20:46:01] Started run 408 on task 146822, dataset 'segment'.
[2018-04-11 20:46:04] Error in run 401: Run already exists in server. Run id(s): {8960510}
[2018-04-11 20:46:04] Started run 409 on task 146822, dataset 'segment'.
[2018-04-11 20:46:04] Error in run 402: Run already exists in server. Run id(s): {8960512}
[2018-04-11 20:46:04] Error in run 403: Run already exists in server. Run id(s): {8960513}
[2018-04-11 20:46:04] Started run 410 on task 146822, dataset 'segment'.
[2018-04-11 20:46:04] Started run 411 on task 146822, dataset 'segment'.
[2018-04-11 20:46:05] Error in run 405: Run already exists in server. Run id(s): {8960516}
[2018-04-11 20:46:05] Started run 412 on task 146822, dataset 'segment'.
[2018-04-11 20:46:05] Error in run 404: Run already exists in server. Run id(s): {8960515}
[2018-04-11 20:46:05] Started run 413 on task 146822, dataset 'segment'.
[2018-04-11 20:46:06] Error in run 406: Run already exists in server. Run id(s): {8960517}
[2018-04-11 20:46:06] Started run 414 on task 146822, dataset 'segment'.
[2018-04-11 20:46:06] Error in run 408: Run already exists in server. Run id(s): {8960521}
[2018-04-11 20:46:06] Started run 415 on task 146822, dataset 'segment'.
[2018-04-11 20:46:06] Error in run 407: Run already exists in server. Run id(s): {8960520}
[2018-04-11 20:46:06] Started run 416 on task 146822, dataset 'segment'.
[2018-04-11 20:46:09] Error in run 409: Run already exists in server. Run id(s): {8960522}
[2018-04-11 20:46:09] Error in run 410: Run already exists in server. Run id(s): {8960523}
[2018-04-11 20:46:09] Started run 417 on task 146822, dataset 'segment'.
[2018-04-11 20:46:09] Started run 418 on task 146822, dataset 'segment'.
[2018-04-11 20:46:09] Error in run 411: Run already exists in server. Run id(s): {8960525}
[2018-04-11 20:46:09] Started run 419 on task 146822, dataset 'segment'.
[2018-04-11 20:46:10] Error in run 412: Run already exists in server. Run id(s): {8960527}
[2018-04-11 20:46:10] Started run 420 on task 146822, dataset 'segment'.
[2018-04-11 20:46:10] Error in run 413: Run already exists in server. Run id(s): {8960528}
[2018-04-11 20:46:10] Started run 421 on task 146822, dataset 'segment'.
[2018-04-11 20:46:10] Error in run 414: Run already exists in server. Run id(s): {8960529}
[2018-04-11 20:46:10] Started run 422 on task 146822, dataset 'segment'.
[2018-04-11 20:46:11] Error in run 416: Run already exists in server. Run id(s): {8960534}
[2018-04-11 20:46:11] Error in run 415: Run already exists in server. Run id(s): {8960532}
[2018-04-11 20:46:11] Started run 423 on task 146822, dataset 'segment'.
[2018-04-11 20:46:11] Started run 424 on task 146822, dataset 'segment'.
[2018-04-11 20:46:14] Error in run 417: Run already exists in server. Run id(s): {8960533}
[2018-04-11 20:46:14] Started run 425 on task 146822, dataset 'segment'.
[2018-04-11 20:46:14] Error in run 419: Run already exists in server. Run id(s): {8960537}
[2018-04-11 20:46:14] Started run 426 on task 146822, dataset 'segment'.
[2018-04-11 20:46:14] Error in run 418: Run already exists in server. Run id(s): {8960536}
[2018-04-11 20:46:14] Started run 427 on task 146822, dataset 'segment'.
[2018-04-11 20:46:14] Error in run 420: Run already exists in server. Run id(s): {8960540}
[2018-04-11 20:46:14] Started run 428 on task 146822, dataset 'segment'.
[2018-04-11 20:46:15] Error in run 421: Run already exists in server. Run id(s): {8960539}
[2018-04-11 20:46:15] Started run 429 on task 146822, dataset 'segment'.
[2018-04-11 20:46:15] Error in run 422: Run already exists in server. Run id(s): {8960541}
[2018-04-11 20:46:15] Started run 430 on task 146822, dataset 'segment'.
[2018-04-11 20:46:15] Error in run 423: Run already exists in server. Run id(s): {8960543}
[2018-04-11 20:46:15] Started run 431 on task 146822, dataset 'segment'.
[2018-04-11 20:46:15] Error in run 424: Run already exists in server. Run id(s): {8960545}
[2018-04-11 20:46:15] Started run 432 on task 146822, dataset 'segment'.
[2018-04-11 20:46:19] Error in run 427: Run already exists in server. Run id(s): {8960549}
[2018-04-11 20:46:19] Error in run 425: Run already exists in server. Run id(s): {8960546}
[2018-04-11 20:46:19] Started run 433 on task 146822, dataset 'segment'.
[2018-04-11 20:46:19] Error in run 426: Run already exists in server. Run id(s): {8960547}
[2018-04-11 20:46:19] Started run 435 on task 146822, dataset 'segment'.
[2018-04-11 20:46:19] Started run 434 on task 146822, dataset 'segment'.
[2018-04-11 20:46:19] Error in run 428: Run already exists in server. Run id(s): {8960550}
[2018-04-11 20:46:19] Started run 436 on task 146822, dataset 'segment'.
[2018-04-11 20:46:19] Error in run 429: Run already exists in server. Run id(s): {8960552}
[2018-04-11 20:46:19] Started run 437 on task 146822, dataset 'segment'.
[2018-04-11 20:46:19] Error in run 430: Run already exists in server. Run id(s): {8960553}
[2018-04-11 20:46:19] Started run 438 on task 146822, dataset 'segment'.
[2018-04-11 20:46:20] Error in run 432: Run already exists in server. Run id(s): {8960555}
[2018-04-11 20:46:20] Started run 439 on task 146822, dataset 'segment'.
[2018-04-11 20:46:20] Error in run 431: Run already exists in server. Run id(s): {8960556}
[2018-04-11 20:46:20] Started run 440 on task 146822, dataset 'segment'.
[2018-04-11 20:46:23] Error in run 433: Run already exists in server. Run id(s): {8960558}
[2018-04-11 20:46:23] Started run 441 on task 146822, dataset 'segment'.
[2018-04-11 20:46:23] Error in run 435: Run already exists in server. Run id(s): {8960561}
[2018-04-11 20:46:23] Error in run 434: Run already exists in server. Run id(s): {8960559}
[2018-04-11 20:46:23] Started run 442 on task 146822, dataset 'segment'.
[2018-04-11 20:46:23] Started run 443 on task 146822, dataset 'segment'.
[2018-04-11 20:46:23] Error in run 436: Run already exists in server. Run id(s): {8960562}
[2018-04-11 20:46:23] Started run 444 on task 146822, dataset 'segment'.
[2018-04-11 20:46:24] Error in run 437: Run already exists in server. Run id(s): {8960564}
[2018-04-11 20:46:24] Started run 445 on task 146822, dataset 'segment'.
[2018-04-11 20:46:24] Error in run 438: Run already exists in server. Run id(s): {8960565}
[2018-04-11 20:46:24] Started run 446 on task 146822, dataset 'segment'.
[2018-04-11 20:46:25] Error in run 440: Run already exists in server. Run id(s): {8960569}
[2018-04-11 20:46:25] Started run 447 on task 146822, dataset 'segment'.
[2018-04-11 20:46:25] Error in run 439: Run already exists in server. Run id(s): {8960567}
[2018-04-11 20:46:25] Started run 448 on task 146822, dataset 'segment'.
[2018-04-11 20:46:28] Error in run 441: Run already exists in server. Run id(s): {8960570}
[2018-04-11 20:46:28] Started run 449 on task 146822, dataset 'segment'.
[2018-04-11 20:46:28] Error in run 443: Run already exists in server. Run id(s): {8960573}
[2018-04-11 20:46:28] Error in run 442: Run already exists in server. Run id(s): {8960571}
[2018-04-11 20:46:28] Started run 450 on task 146822, dataset 'segment'.
[2018-04-11 20:46:28] Started run 451 on task 146822, dataset 'segment'.
[2018-04-11 20:46:28] Error in run 444: Run already exists in server. Run id(s): {8960574}
[2018-04-11 20:46:28] Started run 452 on task 146822, dataset 'segment'.
[2018-04-11 20:46:29] Error in run 445: Run already exists in server. Run id(s): {8960576}
[2018-04-11 20:46:29] Started run 453 on task 146822, dataset 'segment'.
[2018-04-11 20:46:29] Error in run 446: Run already exists in server. Run id(s): {8960577}
[2018-04-11 20:46:29] Started run 454 on task 146822, dataset 'segment'.
[2018-04-11 20:46:29] Error in run 447: Run already exists in server. Run id(s): {8960579}
[2018-04-11 20:46:29] Started run 455 on task 146822, dataset 'segment'.
[2018-04-11 20:46:29] Error in run 448: Run already exists in server. Run id(s): {8960580}
[2018-04-11 20:46:29] Started run 456 on task 146822, dataset 'segment'.
[2018-04-11 20:46:33] Error in run 449: Run already exists in server. Run id(s): {8960582}
[2018-04-11 20:46:33] Started run 457 on task 146822, dataset 'segment'.
[2018-04-11 20:46:33] Error in run 451: Run already exists in server. Run id(s): {8960585}
[2018-04-11 20:46:33] Started run 458 on task 146822, dataset 'segment'.
[2018-04-11 20:46:33] Error in run 450: Run already exists in server. Run id(s): {8960584}
[2018-04-11 20:46:33] Started run 459 on task 146822, dataset 'segment'.
[2018-04-11 20:46:33] Error in run 452: Run already exists in server. Run id(s): {8960586}
[2018-04-11 20:46:33] Started run 460 on task 146822, dataset 'segment'.
[2018-04-11 20:46:33] Error in run 453: Run already exists in server. Run id(s): {8960588}
[2018-04-11 20:46:33] Started run 461 on task 146822, dataset 'segment'.
[2018-04-11 20:46:33] Error in run 454: Run already exists in server. Run id(s): {8960589}
[2018-04-11 20:46:33] Started run 462 on task 146822, dataset 'segment'.
[2018-04-11 20:46:34] Error in run 455: Run already exists in server. Run id(s): {8960591}
[2018-04-11 20:46:34] Started run 463 on task 146822, dataset 'segment'.
[2018-04-11 20:46:34] Error in run 456: Run already exists in server. Run id(s): {8960593}
[2018-04-11 20:46:34] Started run 464 on task 146822, dataset 'segment'.
[2018-04-11 20:46:37] Error in run 457: Run already exists in server. Run id(s): {8960594}
[2018-04-11 20:46:37] Started run 465 on task 146822, dataset 'segment'.
[2018-04-11 20:46:37] Error in run 459: Run already exists in server. Run id(s): {8960597}
[2018-04-11 20:46:37] Started run 466 on task 146822, dataset 'segment'.
[2018-04-11 20:46:37] Error in run 458: Run already exists in server. Run id(s): {8960595}
[2018-04-11 20:46:37] Started run 467 on task 146822, dataset 'segment'.
[2018-04-11 20:46:37] Error in run 460: Run already exists in server. Run id(s): {8960598}
[2018-04-11 20:46:37] Started run 468 on task 146822, dataset 'segment'.
[2018-04-11 20:46:38] Error in run 461: Run already exists in server. Run id(s): {8960601}
[2018-04-11 20:46:38] Started run 469 on task 146822, dataset 'segment'.
[2018-04-11 20:46:38] Error in run 462: Run already exists in server. Run id(s): {8960602}
[2018-04-11 20:46:38] Started run 470 on task 146822, dataset 'segment'.
[2018-04-11 20:46:39] Error in run 463: Run already exists in server. Run id(s): {8960600}
[2018-04-11 20:46:39] Started run 471 on task 146822, dataset 'segment'.
[2018-04-11 20:46:39] Error in run 464: Run already exists in server. Run id(s): {8960604}
[2018-04-11 20:46:39] Started run 472 on task 146822, dataset 'segment'.
[2018-04-11 20:46:42] Error in run 465: Run already exists in server. Run id(s): {8960609}
[2018-04-11 20:46:42] Started run 473 on task 146822, dataset 'segment'.
[2018-04-11 20:46:42] Error in run 466: Run already exists in server. Run id(s): {8960607}
[2018-04-11 20:46:42] Started run 474 on task 146822, dataset 'segment'.
[2018-04-11 20:46:42] Error in run 467: Run already exists in server. Run id(s): {8960608}
[2018-04-11 20:46:42] Error in run 468: Run already exists in server. Run id(s): {8960610}
[2018-04-11 20:46:42] Started run 475 on task 146822, dataset 'segment'.
[2018-04-11 20:46:42] Started run 476 on task 146822, dataset 'segment'.
[2018-04-11 20:46:42] Error in run 469: Run already exists in server. Run id(s): {8960614}
[2018-04-11 20:46:42] Started run 477 on task 146822, dataset 'segment'.
[2018-04-11 20:46:43] Error in run 470: Run already exists in server. Run id(s): {8960612}
[2018-04-11 20:46:43] Started run 478 on task 146822, dataset 'segment'.
[2018-04-11 20:46:44] Error in run 471: Run already exists in server. Run id(s): {8960616}
[2018-04-11 20:46:44] Started run 479 on task 146822, dataset 'segment'.
[2018-04-11 20:46:44] Error in run 472: Run already exists in server. Run id(s): {8960615}
[2018-04-11 20:46:44] Started run 480 on task 146822, dataset 'segment'.
[2018-04-11 20:46:46] Error in run 473: Run already exists in server. Run id(s): {8960620}
[2018-04-11 20:46:46] Started run 481 on task 146822, dataset 'segment'.
[2018-04-11 20:46:47] Error in run 477: Run already exists in server. Run id(s): {8960627}
[2018-04-11 20:46:47] Started run 482 on task 146822, dataset 'segment'.
[2018-04-11 20:46:47] Error in run 476: Run already exists in server. Run id(s): {8960622}
[2018-04-11 20:46:47] Error in run 474: Run already exists in server. Run id(s): {8960621}
[2018-04-11 20:46:47] Error in run 475: Run already exists in server. Run id(s): {8960619}
[2018-04-11 20:46:47] Started run 483 on task 146822, dataset 'segment'.
[2018-04-11 20:46:47] Started run 484 on task 146822, dataset 'segment'.
[2018-04-11 20:46:47] Started run 485 on task 146822, dataset 'segment'.
[2018-04-11 20:46:47] Error in run 478: Run already exists in server. Run id(s): {8960624}
[2018-04-11 20:46:47] Started run 486 on task 146822, dataset 'segment'.
[2018-04-11 20:46:48] Error in run 480: Run already exists in server. Run id(s): {8960629}
[2018-04-11 20:46:48] Error in run 479: Run already exists in server. Run id(s): {8960626}
[2018-04-11 20:46:48] Started run 487 on task 146822, dataset 'segment'.
[2018-04-11 20:46:48] Started run 488 on task 146822, dataset 'segment'.
[2018-04-11 20:46:51] Error in run 481: Run already exists in server. Run id(s): {8960631}
[2018-04-11 20:46:51] Started run 489 on task 146822, dataset 'segment'.
[2018-04-11 20:46:51] Error in run 484: Run already exists in server. Run id(s): {8960634}
[2018-04-11 20:46:51] Started run 490 on task 146822, dataset 'segment'.
[2018-04-11 20:46:51] Error in run 485: Run already exists in server. Run id(s): {8960636}
[2018-04-11 20:46:51] Error in run 483: Run already exists in server. Run id(s): {8960633}
[2018-04-11 20:46:51] Error in run 482: Run already exists in server. Run id(s): {8960632}
[2018-04-11 20:46:51] Started run 491 on task 146822, dataset 'segment'.
[2018-04-11 20:46:51] Started run 492 on task 146822, dataset 'segment'.
[2018-04-11 20:46:51] Started run 493 on task 146822, dataset 'segment'.
[2018-04-11 20:46:52] Error in run 486: Run already exists in server. Run id(s): {8960637}
[2018-04-11 20:46:52] Started run 494 on task 146822, dataset 'segment'.
[2018-04-11 20:46:53] Error in run 487: Run already exists in server. Run id(s): {8960641}
[2018-04-11 20:46:53] Started run 495 on task 146822, dataset 'segment'.
[2018-04-11 20:46:53] Error in run 488: Run already exists in server. Run id(s): {8960640}
[2018-04-11 20:46:53] Started run 496 on task 146822, dataset 'segment'.
[2018-04-11 20:46:55] Error in run 489: Run already exists in server. Run id(s): {8960643}
[2018-04-11 20:46:55] Started run 497 on task 146822, dataset 'segment'.
[2018-04-11 20:46:56] Error in run 492: Run already exists in server. Run id(s): {8960647}
[2018-04-11 20:46:56] Started run 498 on task 146822, dataset 'segment'.
[2018-04-11 20:46:56] Error in run 491: Run already exists in server. Run id(s): {8960645}
[2018-04-11 20:46:56] Error in run 490: Run already exists in server. Run id(s): {8960644}
[2018-04-11 20:46:56] Started run 499 on task 146822, dataset 'segment'.
[2018-04-11 20:46:56] Error in run 493: Run already exists in server. Run id(s): {8960648}
[2018-04-11 20:46:56] Started run 501 on task 146822, dataset 'segment'.
[2018-04-11 20:46:56] Started run 500 on task 146822, dataset 'segment'.
[2018-04-11 20:46:56] Error in run 494: Run already exists in server. Run id(s): {8960649}
[2018-04-11 20:46:56] Started run 502 on task 146822, dataset 'segment'.
[2018-04-11 20:46:57] Error in run 495: Run already exists in server. Run id(s): {8960651}
[2018-04-11 20:46:57] Started run 503 on task 146822, dataset 'segment'.
[2018-04-11 20:46:57] Error in run 496: Run already exists in server. Run id(s): {8960653}
[2018-04-11 20:46:57] Started run 504 on task 146822, dataset 'segment'.
[2018-04-11 20:46:59] Error in run 497: Run already exists in server. Run id(s): {8960654}
[2018-04-11 20:46:59] Started run 505 on task 146822, dataset 'segment'.
[2018-04-11 20:47:00] Error in run 498: Run already exists in server. Run id(s): {8960655}
[2018-04-11 20:47:00] Started run 506 on task 146822, dataset 'segment'.
[2018-04-11 20:47:01] Error in run 499: Run already exists in server. Run id(s): {8960657}
[2018-04-11 20:47:01] Started run 507 on task 146822, dataset 'segment'.
[2018-04-11 20:47:01] Error in run 501: Run already exists in server. Run id(s): {8960660}
[2018-04-11 20:47:01] Started run 508 on task 146822, dataset 'segment'.
[2018-04-11 20:47:01] Error in run 500: Run already exists in server. Run id(s): {8960659}
[2018-04-11 20:47:01] Started run 509 on task 146822, dataset 'segment'.
[2018-04-11 20:47:01] Error in run 502: Run already exists in server. Run id(s): {8960661}
[2018-04-11 20:47:01] Started run 510 on task 146822, dataset 'segment'.
[2018-04-11 20:47:02] Error in run 503: Run already exists in server. Run id(s): {8960662}
[2018-04-11 20:47:02] Started run 511 on task 146822, dataset 'segment'.
[2018-04-11 20:47:02] Error in run 504: Run already exists in server. Run id(s): {8960665}
[2018-04-11 20:47:02] Started run 512 on task 146822, dataset 'segment'.
[2018-04-11 20:47:04] Error in run 505: Run already exists in server. Run id(s): {8960668}
[2018-04-11 20:47:04] Started run 513 on task 146822, dataset 'segment'.
[2018-04-11 20:47:05] Error in run 506: Run already exists in server. Run id(s): {8960667}
[2018-04-11 20:47:05] Started run 514 on task 146822, dataset 'segment'.
[2018-04-11 20:47:05] Error in run 507: Run already exists in server. Run id(s): {8960669}
[2018-04-11 20:47:05] Started run 515 on task 146822, dataset 'segment'.
[2018-04-11 20:47:05] Error in run 508: Run already exists in server. Run id(s): {8960671}
[2018-04-11 20:47:05] Started run 516 on task 146822, dataset 'segment'.
[2018-04-11 20:47:05] Error in run 509: Run already exists in server. Run id(s): {8960673}
[2018-04-11 20:47:05] Error in run 510: Run already exists in server. Run id(s): {8960674}
[2018-04-11 20:47:05] Started run 517 on task 146822, dataset 'segment'.
[2018-04-11 20:47:05] Started run 518 on task 146822, dataset 'segment'.
[2018-04-11 20:47:06] Error in run 511: Run already exists in server. Run id(s): {8960672}
[2018-04-11 20:47:06] Started run 519 on task 146822, dataset 'segment'.
[2018-04-11 20:47:06] Error in run 512: Run already exists in server. Run id(s): {8960677}
[2018-04-11 20:47:06] Started run 520 on task 146822, dataset 'segment'.
[2018-04-11 20:47:09] Error in run 513: Run already exists in server. Run id(s): {8960679}
[2018-04-11 20:47:09] Started run 521 on task 146822, dataset 'segment'.
[2018-04-11 20:47:09] Error in run 514: Run already exists in server. Run id(s): {8960680}
[2018-04-11 20:47:09] Started run 522 on task 146822, dataset 'segment'.
[2018-04-11 20:47:09] Error in run 515: Run already exists in server. Run id(s): {8960681}
[2018-04-11 20:47:09] Started run 523 on task 146822, dataset 'segment'.
[2018-04-11 20:47:10] Error in run 516: Run already exists in server. Run id(s): {8960683}
[2018-04-11 20:47:10] Started run 524 on task 146822, dataset 'segment'.
[2018-04-11 20:47:10] Error in run 517: Run already exists in server. Run id(s): {8960686}
[2018-04-11 20:47:10] Started run 525 on task 146822, dataset 'segment'.
[2018-04-11 20:47:10] Error in run 518: Run already exists in server. Run id(s): {8960687}
[2018-04-11 20:47:10] Started run 526 on task 146822, dataset 'segment'.
[2018-04-11 20:47:11] Error in run 519: Run already exists in server. Run id(s): {8960684}
[2018-04-11 20:47:11] Started run 527 on task 146822, dataset 'segment'.
[2018-04-11 20:47:11] Error in run 520: Run already exists in server. Run id(s): {8960689}
[2018-04-11 20:47:11] Started run 528 on task 146822, dataset 'segment'.
[2018-04-11 20:47:13] Error in run 521: Run already exists in server. Run id(s): {8960691}
[2018-04-11 20:47:13] Started run 529 on task 146822, dataset 'segment'.
[2018-04-11 20:47:14] Error in run 522: Run already exists in server. Run id(s): {8960692}
[2018-04-11 20:47:14] Started run 530 on task 146822, dataset 'segment'.
[2018-04-11 20:47:14] Error in run 523: Run already exists in server. Run id(s): {8960693}
[2018-04-11 20:47:14] Started run 531 on task 146822, dataset 'segment'.
[2018-04-11 20:47:14] Error in run 524: Run already exists in server. Run id(s): {8960695}
[2018-04-11 20:47:14] Started run 532 on task 146822, dataset 'segment'.
[2018-04-11 20:47:15] Error in run 525: Run already exists in server. Run id(s): {8960696}
[2018-04-11 20:47:15] Started run 533 on task 146822, dataset 'segment'.
[2018-04-11 20:47:15] Error in run 526: Run already exists in server. Run id(s): {8960697}
[2018-04-11 20:47:15] Started run 534 on task 146822, dataset 'segment'.
[2018-04-11 20:47:16] Error in run 527: Run already exists in server. Run id(s): {8960699}
[2018-04-11 20:47:16] Started run 535 on task 146822, dataset 'segment'.
[2018-04-11 20:47:16] Error in run 528: Run already exists in server. Run id(s): {8960701}
[2018-04-11 20:47:16] Started run 536 on task 146822, dataset 'segment'.
[2018-04-11 20:47:18] Error in run 529: Run already exists in server. Run id(s): {8960703}
[2018-04-11 20:47:18] Started run 537 on task 146822, dataset 'segment'.
[2018-04-11 20:47:19] Error in run 530: Run already exists in server. Run id(s): {8960704}
[2018-04-11 20:47:19] Started run 538 on task 146822, dataset 'segment'.
[2018-04-11 20:47:19] Error in run 531: Run already exists in server. Run id(s): {8960705}
[2018-04-11 20:47:19] Started run 539 on task 146822, dataset 'segment'.
[2018-04-11 20:47:19] Error in run 532: Run already exists in server. Run id(s): {8960706}
[2018-04-11 20:47:19] Started run 540 on task 146822, dataset 'segment'.
[2018-04-11 20:47:19] Error in run 533: Run already exists in server. Run id(s): {8960709}
[2018-04-11 20:47:19] Started run 541 on task 146822, dataset 'segment'.
[2018-04-11 20:47:19] Error in run 534: Run already exists in server. Run id(s): {8960708}
[2018-04-11 20:47:19] Started run 542 on task 146822, dataset 'segment'.
[2018-04-11 20:47:20] Error in run 535: Run already exists in server. Run id(s): {8960710}
[2018-04-11 20:47:20] Started run 543 on task 146822, dataset 'segment'.
[2018-04-11 20:47:20] Error in run 536: Run already exists in server. Run id(s): {8960712}
[2018-04-11 20:47:20] Started run 544 on task 146822, dataset 'segment'.
[2018-04-11 20:47:23] Error in run 537: Run already exists in server. Run id(s): {8960714}
[2018-04-11 20:47:23] Started run 545 on task 146822, dataset 'segment'.
[2018-04-11 20:47:23] Error in run 539: Run already exists in server. Run id(s): {8960717}
[2018-04-11 20:47:23] Started run 546 on task 146822, dataset 'segment'.
[2018-04-11 20:47:23] Error in run 540: Run already exists in server. Run id(s): {8960718}
[2018-04-11 20:47:23] Started run 547 on task 146822, dataset 'segment'.
[2018-04-11 20:47:23] Error in run 538: Run already exists in server. Run id(s): {8960716}
[2018-04-11 20:47:23] Started run 548 on task 146822, dataset 'segment'.
[2018-04-11 20:47:24] Error in run 541: Run already exists in server. Run id(s): {8960720}
[2018-04-11 20:47:24] Started run 549 on task 146822, dataset 'segment'.
[2018-04-11 20:47:24] Error in run 542: Run already exists in server. Run id(s): {8960723}
[2018-04-11 20:47:24] Started run 550 on task 146822, dataset 'segment'.
[2018-04-11 20:47:25] Error in run 543: Run already exists in server. Run id(s): {8960721}
[2018-04-11 20:47:25] Started run 551 on task 146822, dataset 'segment'.
[2018-04-11 20:47:25] Error in run 544: Run already exists in server. Run id(s): {8960724}
[2018-04-11 20:47:25] Started run 552 on task 146822, dataset 'segment'.
[2018-04-11 20:47:27] Error in run 545: Run already exists in server. Run id(s): {8960726}
[2018-04-11 20:47:27] Started run 553 on task 146822, dataset 'segment'.
[2018-04-11 20:47:28] Error in run 546: Run already exists in server. Run id(s): {8960728}
[2018-04-11 20:47:28] Started run 554 on task 146822, dataset 'segment'.
[2018-04-11 20:47:28] Error in run 547: Run already exists in server. Run id(s): {8960729}
[2018-04-11 20:47:28] Started run 555 on task 146822, dataset 'segment'.
[2018-04-11 20:47:28] Error in run 548: Run already exists in server. Run id(s): {8960730}
[2018-04-11 20:47:28] Started run 556 on task 146822, dataset 'segment'.
[2018-04-11 20:47:28] Error in run 549: Run already exists in server. Run id(s): {8960732}
[2018-04-11 20:47:28] Started run 557 on task 146822, dataset 'segment'.
[2018-04-11 20:47:29] Error in run 550: Run already exists in server. Run id(s): {8960733}
[2018-04-11 20:47:29] Started run 558 on task 146822, dataset 'segment'.
[2018-04-11 20:47:29] Error in run 551: Run already exists in server. Run id(s): {8960734}
[2018-04-11 20:47:29] Started run 559 on task 146822, dataset 'segment'.
[2018-04-11 20:47:30] Error in run 552: Run already exists in server. Run id(s): {8960736}
[2018-04-11 20:47:30] Started run 560 on task 146822, dataset 'segment'.
[2018-04-11 20:47:32] Error in run 553: Run already exists in server. Run id(s): {8960740}
[2018-04-11 20:47:32] Started run 561 on task 146822, dataset 'segment'.
[2018-04-11 20:47:32] Error in run 554: Run already exists in server. Run id(s): {8960739}
[2018-04-11 20:47:32] Started run 562 on task 146822, dataset 'segment'.
[2018-04-11 20:47:33] Error in run 555: Run already exists in server. Run id(s): {8960741}
[2018-04-11 20:47:33] Started run 563 on task 146822, dataset 'segment'.
[2018-04-11 20:47:33] Error in run 556: Run already exists in server. Run id(s): {8960742}
[2018-04-11 20:47:33] Started run 564 on task 146822, dataset 'segment'.
[2018-04-11 20:47:33] Error in run 557: Run already exists in server. Run id(s): {8960744}
[2018-04-11 20:47:33] Started run 565 on task 146822, dataset 'segment'.
[2018-04-11 20:47:34] Error in run 558: Run already exists in server. Run id(s): {8960746}
[2018-04-11 20:47:34] Started run 566 on task 146822, dataset 'segment'.
[2018-04-11 20:47:34] Error in run 559: Run already exists in server. Run id(s): {8960745}
[2018-04-11 20:47:34] Started run 567 on task 146822, dataset 'segment'.
[2018-04-11 20:47:34] Error in run 560: Run already exists in server. Run id(s): {8960747}
[2018-04-11 20:47:34] Started run 568 on task 146822, dataset 'segment'.
[2018-04-11 20:47:36] Error in run 561: Run already exists in server. Run id(s): {8960750}
[2018-04-11 20:47:36] Started run 569 on task 146822, dataset 'segment'.
[2018-04-11 20:47:37] Error in run 562: Run already exists in server. Run id(s): {8960753}
[2018-04-11 20:47:37] Started run 570 on task 146822, dataset 'segment'.
[2018-04-11 20:47:37] Error in run 563: Run already exists in server. Run id(s): {8960752}
[2018-04-11 20:47:37] Started run 571 on task 146822, dataset 'segment'.
[2018-04-11 20:47:37] Error in run 565: Run already exists in server. Run id(s): {8960756}
[2018-04-11 20:47:37] Error in run 564: Run already exists in server. Run id(s): {8960754}
[2018-04-11 20:47:37] Started run 572 on task 146822, dataset 'segment'.
[2018-04-11 20:47:37] Started run 573 on task 146822, dataset 'segment'.
[2018-04-11 20:47:38] Error in run 566: Run already exists in server. Run id(s): {8960757}
[2018-04-11 20:47:38] Started run 574 on task 146822, dataset 'segment'.
[2018-04-11 20:47:38] Error in run 567: Run already exists in server. Run id(s): {8960760}
[2018-04-11 20:47:38] Started run 575 on task 146822, dataset 'segment'.
[2018-04-11 20:47:39] Error in run 568: Run already exists in server. Run id(s): {8960759}
[2018-04-11 20:47:39] Started run 576 on task 146822, dataset 'segment'.
[2018-04-11 20:47:41] Error in run 569: Run already exists in server. Run id(s): {8960762}
[2018-04-11 20:47:41] Started run 577 on task 146822, dataset 'segment'.
[2018-04-11 20:47:41] Error in run 570: Run already exists in server. Run id(s): {8960763}
[2018-04-11 20:47:41] Started run 578 on task 146822, dataset 'segment'.
[2018-04-11 20:47:42] Error in run 571: Run already exists in server. Run id(s): {8960766}
[2018-04-11 20:47:42] Started run 579 on task 146822, dataset 'segment'.
[2018-04-11 20:47:42] Error in run 573: Run already exists in server. Run id(s): {8960769}
[2018-04-11 20:47:42] Started run 580 on task 146822, dataset 'segment'.
[2018-04-11 20:47:42] Error in run 572: Run already exists in server. Run id(s): {8960765}
[2018-04-11 20:47:42] Started run 581 on task 146822, dataset 'segment'.
[2018-04-11 20:47:43] Error in run 574: Run already exists in server. Run id(s): {8960768}
[2018-04-11 20:47:43] Started run 582 on task 146822, dataset 'segment'.
[2018-04-11 20:47:43] Error in run 575: Run already exists in server. Run id(s): {8960771}
[2018-04-11 20:47:43] Started run 583 on task 146822, dataset 'segment'.
[2018-04-11 20:47:44] Error in run 576: Run already exists in server. Run id(s): {8960772}
[2018-04-11 20:47:44] Started run 584 on task 146822, dataset 'segment'.
[2018-04-11 20:47:45] Error in run 577: Run already exists in server. Run id(s): {8960774}
[2018-04-11 20:47:45] Started run 585 on task 146822, dataset 'segment'.
[2018-04-11 20:47:46] Error in run 578: Run already exists in server. Run id(s): {8960775}
[2018-04-11 20:47:46] Started run 586 on task 146822, dataset 'segment'.
[2018-04-11 20:47:46] Error in run 579: Run already exists in server. Run id(s): {8960777}
[2018-04-11 20:47:46] Started run 587 on task 146822, dataset 'segment'.
[2018-04-11 20:47:46] Error in run 580: Run already exists in server. Run id(s): {8960778}
[2018-04-11 20:47:46] Started run 588 on task 146822, dataset 'segment'.
[2018-04-11 20:47:47] Error in run 581: Run already exists in server. Run id(s): {8960780}
[2018-04-11 20:47:47] Started run 589 on task 146822, dataset 'segment'.
[2018-04-11 20:47:47] Error in run 582: Run already exists in server. Run id(s): {8960781}
[2018-04-11 20:47:47] Started run 590 on task 146822, dataset 'segment'.
[2018-04-11 20:47:47] Error in run 583: Run already exists in server. Run id(s): {8960782}
[2018-04-11 20:47:47] Started run 591 on task 146822, dataset 'segment'.
[2018-04-11 20:47:48] Error in run 584: Run already exists in server. Run id(s): {8960784}
[2018-04-11 20:47:48] Started run 592 on task 146822, dataset 'segment'.
[2018-04-11 20:47:50] Error in run 585: Run already exists in server. Run id(s): {8960786}
[2018-04-11 20:47:50] Started run 593 on task 146822, dataset 'segment'.
[2018-04-11 20:47:51] Error in run 586: Run already exists in server. Run id(s): {8960788}
[2018-04-11 20:47:51] Started run 594 on task 146822, dataset 'segment'.
[2018-04-11 20:47:51] Error in run 587: Run already exists in server. Run id(s): {8960787}
[2018-04-11 20:47:51] Started run 595 on task 146822, dataset 'segment'.
[2018-04-11 20:47:51] Error in run 588: Run already exists in server. Run id(s): {8960790}
[2018-04-11 20:47:51] Started run 596 on task 146822, dataset 'segment'.
[2018-04-11 20:47:51] Error in run 589: Run already exists in server. Run id(s): {8960792}
[2018-04-11 20:47:51] Started run 597 on task 146822, dataset 'segment'.
[2018-04-11 20:47:52] Error in run 590: Run already exists in server. Run id(s): {8960793}
[2018-04-11 20:47:52] Started run 598 on task 146822, dataset 'segment'.
[2018-04-11 20:47:52] Error in run 591: Run already exists in server. Run id(s): {8960794}
[2018-04-11 20:47:52] Started run 599 on task 146822, dataset 'segment'.
[2018-04-11 20:47:53] Error in run 592: Run already exists in server. Run id(s): {8960795}
[2018-04-11 20:47:53] Started run 600 on task 146822, dataset 'segment'.
[2018-04-11 20:47:54] Error in run 593: Run already exists in server. Run id(s): {8960797}
[2018-04-11 20:47:54] Started run 601 on task 146822, dataset 'segment'.
[2018-04-11 20:47:55] Error in run 594: Run already exists in server. Run id(s): {8960799}
[2018-04-11 20:47:55] Started run 602 on task 146822, dataset 'segment'.
[2018-04-11 20:47:55] Error in run 595: Run already exists in server. Run id(s): {8960800}
[2018-04-11 20:47:55] Started run 603 on task 146822, dataset 'segment'.
[2018-04-11 20:47:55] Error in run 596: Run already exists in server. Run id(s): {8960802}
[2018-04-11 20:47:55] Started run 604 on task 146822, dataset 'segment'.
[2018-04-11 20:47:56] Error in run 597: Run already exists in server. Run id(s): {8960803}
[2018-04-11 20:47:56] Started run 605 on task 146822, dataset 'segment'.
[2018-04-11 20:47:56] Error in run 598: Run already exists in server. Run id(s): {8960805}
[2018-04-11 20:47:56] Started run 606 on task 146822, dataset 'segment'.
[2018-04-11 20:47:56] Error in run 599: Run already exists in server. Run id(s): {8960806}
[2018-04-11 20:47:56] Started run 607 on task 146822, dataset 'segment'.
[2018-04-11 20:47:57] Error in run 600: Run already exists in server. Run id(s): {8960808}
[2018-04-11 20:47:57] Started run 608 on task 146822, dataset 'segment'.
[2018-04-11 20:47:59] Error in run 601: Run already exists in server. Run id(s): {8960809}
[2018-04-11 20:47:59] Started run 609 on task 146822, dataset 'segment'.
[2018-04-11 20:48:00] Error in run 602: Run already exists in server. Run id(s): {8960810}
[2018-04-11 20:48:00] Started run 610 on task 146822, dataset 'segment'.
[2018-04-11 20:48:00] Error in run 603: Run already exists in server. Run id(s): {8960812}
[2018-04-11 20:48:00] Started run 611 on task 146822, dataset 'segment'.
[2018-04-11 20:48:00] Error in run 604: Run already exists in server. Run id(s): {8960814}
[2018-04-11 20:48:00] Started run 612 on task 146822, dataset 'segment'.
[2018-04-11 20:48:00] Error in run 605: Run already exists in server. Run id(s): {8960816}
[2018-04-11 20:48:00] Started run 613 on task 146822, dataset 'segment'.
[2018-04-11 20:48:01] Error in run 606: Run already exists in server. Run id(s): {8960815}
[2018-04-11 20:48:01] Started run 614 on task 146822, dataset 'segment'.
[2018-04-11 20:48:01] Error in run 607: Run already exists in server. Run id(s): {8960818}
[2018-04-11 20:48:01] Started run 615 on task 146822, dataset 'segment'.
[2018-04-11 20:48:02] Error in run 608: Run already exists in server. Run id(s): {8960820}
[2018-04-11 20:48:02] Started run 616 on task 146822, dataset 'segment'.
[2018-04-11 20:48:04] Error in run 609: Run already exists in server. Run id(s): {8960821}
[2018-04-11 20:48:04] Started run 617 on task 146822, dataset 'segment'.
[2018-04-11 20:48:04] Error in run 610: Run already exists in server. Run id(s): {8960823}
[2018-04-11 20:48:04] Started run 618 on task 146822, dataset 'segment'.
[2018-04-11 20:48:04] Error in run 611: Run already exists in server. Run id(s): {8960824}
[2018-04-11 20:48:04] Started run 619 on task 146822, dataset 'segment'.
[2018-04-11 20:48:05] Error in run 612: Run already exists in server. Run id(s): {8960826}
[2018-04-11 20:48:05] Started run 620 on task 146822, dataset 'segment'.
[2018-04-11 20:48:05] Error in run 613: Run already exists in server. Run id(s): {8960827}
[2018-04-11 20:48:05] Started run 621 on task 146822, dataset 'segment'.
[2018-04-11 20:48:05] Error in run 614: Run already exists in server. Run id(s): {8960830}
[2018-04-11 20:48:05] Started run 622 on task 146822, dataset 'segment'.
[2018-04-11 20:48:06] Error in run 615: Run already exists in server. Run id(s): {8960829}
[2018-04-11 20:48:06] Started run 623 on task 146822, dataset 'segment'.
[2018-04-11 20:48:06] Error in run 616: Run already exists in server. Run id(s): {8960832}
[2018-04-11 20:48:06] Started run 624 on task 146822, dataset 'segment'.
[2018-04-11 20:48:08] Error in run 617: Run already exists in server. Run id(s): {8960834}
[2018-04-11 20:48:08] Started run 625 on task 146822, dataset 'segment'.
[2018-04-11 20:48:09] Error in run 618: Run already exists in server. Run id(s): {8960836}
[2018-04-11 20:48:09] Started run 626 on task 146822, dataset 'segment'.
[2018-04-11 20:48:09] Error in run 619: Run already exists in server. Run id(s): {8960835}
[2018-04-11 20:48:09] Started run 627 on task 146822, dataset 'segment'.
[2018-04-11 20:48:09] Error in run 620: Run already exists in server. Run id(s): {8960838}
[2018-04-11 20:48:09] Started run 628 on task 146822, dataset 'segment'.
[2018-04-11 20:48:09] Error in run 621: Run already exists in server. Run id(s): {8960840}
[2018-04-11 20:48:09] Started run 629 on task 146822, dataset 'segment'.
[2018-04-11 20:48:10] Error in run 622: Run already exists in server. Run id(s): {8960842}
[2018-04-11 20:48:10] Started run 630 on task 146822, dataset 'segment'.
[2018-04-11 20:48:10] Error in run 623: Run already exists in server. Run id(s): {8960841}
[2018-04-11 20:48:10] Started run 631 on task 146822, dataset 'segment'.
[2018-04-11 20:48:11] Error in run 624: Run already exists in server. Run id(s): {8960844}
[2018-04-11 20:48:11] Started run 632 on task 146822, dataset 'segment'.
[2018-04-11 20:48:13] Error in run 625: Run already exists in server. Run id(s): {8960846}
[2018-04-11 20:48:13] Started run 633 on task 146822, dataset 'segment'.
[2018-04-11 20:48:13] Error in run 626: Run already exists in server. Run id(s): {8960847}
[2018-04-11 20:48:13] Started run 634 on task 146822, dataset 'segment'.
[2018-04-11 20:48:13] Error in run 627: Run already exists in server. Run id(s): {8960848}
[2018-04-11 20:48:13] Started run 635 on task 146822, dataset 'segment'.
[2018-04-11 20:48:14] Error in run 628: Run already exists in server. Run id(s): {8960849}
[2018-04-11 20:48:14] Started run 636 on task 146822, dataset 'segment'.
[2018-04-11 20:48:14] Error in run 629: Run already exists in server. Run id(s): {8960852}
[2018-04-11 20:48:14] Started run 637 on task 146822, dataset 'segment'.
[2018-04-11 20:48:14] Error in run 630: Run already exists in server. Run id(s): {8960853}
[2018-04-11 20:48:14] Started run 638 on task 146822, dataset 'segment'.
[2018-04-11 20:48:15] Error in run 631: Run already exists in server. Run id(s): {8960854}
[2018-04-11 20:48:15] Started run 639 on task 146822, dataset 'segment'.
[2018-04-11 20:48:16] Error in run 632: Run already exists in server. Run id(s): {8960856}
[2018-04-11 20:48:16] Started run 640 on task 146822, dataset 'segment'.
[2018-04-11 20:48:17] Error in run 633: Run already exists in server. Run id(s): {8960858}
[2018-04-11 20:48:17] Started run 641 on task 146822, dataset 'segment'.
[2018-04-11 20:48:18] Error in run 634: Run already exists in server. Run id(s): {8960859}
[2018-04-11 20:48:18] Started run 642 on task 146822, dataset 'segment'.
[2018-04-11 20:48:18] Error in run 635: Run already exists in server. Run id(s): {8960860}
[2018-04-11 20:48:18] Started run 643 on task 146822, dataset 'segment'.
[2018-04-11 20:48:18] Error in run 636: Run already exists in server. Run id(s): {8960861}
[2018-04-11 20:48:18] Started run 644 on task 146822, dataset 'segment'.
[2018-04-11 20:48:18] Error in run 637: Run already exists in server. Run id(s): {8960863}
[2018-04-11 20:48:18] Started run 645 on task 146822, dataset 'segment'.
[2018-04-11 20:48:19] Error in run 638: Run already exists in server. Run id(s): {8960864}
[2018-04-11 20:48:19] Started run 646 on task 146822, dataset 'segment'.
[2018-04-11 20:48:20] Error in run 639: Run already exists in server. Run id(s): {8960865}
[2018-04-11 20:48:20] Started run 647 on task 146822, dataset 'segment'.
[2018-04-11 20:48:20] Error in run 640: Run already exists in server. Run id(s): {8960867}
[2018-04-11 20:48:20] Started run 648 on task 146822, dataset 'segment'.
[2018-04-11 20:48:22] Error in run 641: Run already exists in server. Run id(s): {8960872}
[2018-04-11 20:48:22] Started run 649 on task 146822, dataset 'segment'.
[2018-04-11 20:48:23] Error in run 642: Run already exists in server. Run id(s): {8960869}
[2018-04-11 20:48:23] Started run 650 on task 146822, dataset 'segment'.
[2018-04-11 20:48:23] Error in run 643: Run already exists in server. Run id(s): {8960870}
[2018-04-11 20:48:23] Started run 651 on task 146822, dataset 'segment'.
[2018-04-11 20:48:23] Error in run 644: Run already exists in server. Run id(s): {8960873}
[2018-04-11 20:48:23] Started run 652 on task 146822, dataset 'segment'.
[2018-04-11 20:48:23] Error in run 645: Run already exists in server. Run id(s): {8960875}
[2018-04-11 20:48:23] Started run 653 on task 146822, dataset 'segment'.
[2018-04-11 20:48:24] Error in run 646: Run already exists in server. Run id(s): {8960876}
[2018-04-11 20:48:24] Started run 654 on task 146822, dataset 'segment'.
[2018-04-11 20:48:24] Error in run 647: Run already exists in server. Run id(s): {8960877}
[2018-04-11 20:48:24] Started run 655 on task 146822, dataset 'segment'.
[2018-04-11 20:48:25] Error in run 648: Run already exists in server. Run id(s): {8960879}
[2018-04-11 20:48:25] Started run 656 on task 146822, dataset 'segment'.
[2018-04-11 20:48:26] Error in run 649: Run already exists in server. Run id(s): {8960881}
[2018-04-11 20:48:26] Started run 657 on task 146822, dataset 'segment'.
[2018-04-11 20:48:27] Error in run 650: Run already exists in server. Run id(s): {8960882}
[2018-04-11 20:48:27] Error in run 651: Run already exists in server. Run id(s): {8960883}
[2018-04-11 20:48:27] Started run 658 on task 146822, dataset 'segment'.
[2018-04-11 20:48:27] Started run 659 on task 146822, dataset 'segment'.
[2018-04-11 20:48:27] Error in run 652: Run already exists in server. Run id(s): {8960885}
[2018-04-11 20:48:27] Started run 660 on task 146822, dataset 'segment'.
[2018-04-11 20:48:27] Error in run 653: Run already exists in server. Run id(s): {9015819, 8960887}
[2018-04-11 20:48:27] Started run 661 on task 146822, dataset 'segment'.
[2018-04-11 20:48:28] Error in run 654: Run already exists in server. Run id(s): {8960888}
[2018-04-11 20:48:28] Started run 662 on task 146822, dataset 'segment'.
[2018-04-11 20:48:29] Error in run 655: Run already exists in server. Run id(s): {8960889}
[2018-04-11 20:48:29] Started run 663 on task 146822, dataset 'segment'.
[2018-04-11 20:48:29] Error in run 656: Run already exists in server. Run id(s): {8960891}
[2018-04-11 20:48:29] Started run 664 on task 146822, dataset 'segment'.
[2018-04-11 20:48:31] Error in run 657: Run already exists in server. Run id(s): {8960894}
[2018-04-11 20:48:31] Started run 665 on task 146822, dataset 'segment'.
[2018-04-11 20:48:32] Error in run 659: Run already exists in server. Run id(s): {8960896}
[2018-04-11 20:48:32] Started run 666 on task 146822, dataset 'segment'.
[2018-04-11 20:48:32] Error in run 658: Run already exists in server. Run id(s): {8960893}
[2018-04-11 20:48:32] Started run 667 on task 146822, dataset 'segment'.
[2018-04-11 20:48:32] Error in run 660: Run already exists in server. Run id(s): {8960897}
[2018-04-11 20:48:32] Error in run 661: Run already exists in server. Run id(s): {8960899}
[2018-04-11 20:48:32] Started run 668 on task 146822, dataset 'segment'.
[2018-04-11 20:48:32] Started run 669 on task 146822, dataset 'segment'.
[2018-04-11 20:48:33] Error in run 662: Run already exists in server. Run id(s): {8960900}
[2018-04-11 20:48:33] Started run 670 on task 146822, dataset 'segment'.
[2018-04-11 20:48:33] Error in run 663: Run already exists in server. Run id(s): {8960902}
[2018-04-11 20:48:33] Started run 671 on task 146822, dataset 'segment'.
[2018-04-11 20:48:34] Error in run 664: Run already exists in server. Run id(s): {8960903}
[2018-04-11 20:48:34] Started run 672 on task 146822, dataset 'segment'.
[2018-04-11 20:48:35] Error in run 665: Run already exists in server. Run id(s): {8960905}
[2018-04-11 20:48:35] Started run 673 on task 146822, dataset 'segment'.
[2018-04-11 20:48:36] Error in run 666: Run already exists in server. Run id(s): {8960906}
[2018-04-11 20:48:36] Started run 674 on task 146822, dataset 'segment'.
[2018-04-11 20:48:36] Error in run 667: Run already exists in server. Run id(s): {8960909}
[2018-04-11 20:48:36] Started run 675 on task 146822, dataset 'segment'.
[2018-04-11 20:48:36] Error in run 668: Run already exists in server. Run id(s): {8960908}
[2018-04-11 20:48:36] Started run 676 on task 146822, dataset 'segment'.
[2018-04-11 20:48:36] Error in run 669: Run already exists in server. Run id(s): {8960911}
[2018-04-11 20:48:36] Started run 677 on task 146822, dataset 'segment'.
[2018-04-11 20:48:37] Error in run 670: Run already exists in server. Run id(s): {8960913}
[2018-04-11 20:48:37] Started run 678 on task 146822, dataset 'segment'.
[2018-04-11 20:48:38] Error in run 671: Run already exists in server. Run id(s): {8960914}
[2018-04-11 20:48:38] Started run 679 on task 146822, dataset 'segment'.
[2018-04-11 20:48:38] Error in run 672: Run already exists in server. Run id(s): {8960915}
[2018-04-11 20:48:38] Started run 680 on task 146822, dataset 'segment'.
[2018-04-11 20:48:40] Error in run 673: Run already exists in server. Run id(s): {8960917}
[2018-04-11 20:48:40] Started run 681 on task 146822, dataset 'segment'.
[2018-04-11 20:48:41] Error in run 674: Run already exists in server. Run id(s): {8960919}
[2018-04-11 20:48:41] Started run 682 on task 146822, dataset 'segment'.
[2018-04-11 20:48:41] Error in run 675: Run already exists in server. Run id(s): {8960921}
[2018-04-11 20:48:41] Started run 683 on task 146822, dataset 'segment'.
[2018-04-11 20:48:41] Error in run 676: Run already exists in server. Run id(s): {8960920}
[2018-04-11 20:48:41] Started run 684 on task 146822, dataset 'segment'.
[2018-04-11 20:48:41] Error in run 677: Run already exists in server. Run id(s): {8960923}
[2018-04-11 20:48:41] Started run 685 on task 146822, dataset 'segment'.
[2018-04-11 20:48:41] Error in run 678: Run already exists in server. Run id(s): {8960926}
[2018-04-11 20:48:41] Started run 686 on task 146822, dataset 'segment'.
[2018-04-11 20:48:42] Error in run 679: Run already exists in server. Run id(s): {8960925}
[2018-04-11 20:48:42] Started run 687 on task 146822, dataset 'segment'.
[2018-04-11 20:48:43] Error in run 680: Run already exists in server. Run id(s): {8960927}
[2018-04-11 20:48:43] Started run 688 on task 146822, dataset 'segment'.
[2018-04-11 20:48:45] Error in run 681: Run already exists in server. Run id(s): {8960929}
[2018-04-11 20:48:45] Started run 689 on task 146822, dataset 'segment'.
[2018-04-11 20:48:45] Error in run 682: Run already exists in server. Run id(s): {8960932}
[2018-04-11 20:48:45] Started run 690 on task 146822, dataset 'segment'.
[2018-04-11 20:48:45] Error in run 684: Run already exists in server. Run id(s): {8960933}
[2018-04-11 20:48:45] Started run 691 on task 146822, dataset 'segment'.
[2018-04-11 20:48:45] Error in run 683: Run already exists in server. Run id(s): {8960931}
[2018-04-11 20:48:45] Started run 692 on task 146822, dataset 'segment'.
[2018-04-11 20:48:45] Error in run 685: Run already exists in server. Run id(s): {8960935}
[2018-04-11 20:48:45] Started run 693 on task 146822, dataset 'segment'.
[2018-04-11 20:48:46] Error in run 686: Run already exists in server. Run id(s): {8960937}
[2018-04-11 20:48:46] Started run 694 on task 146822, dataset 'segment'.
[2018-04-11 20:48:47] Error in run 687: Run already exists in server. Run id(s): {8960938}
[2018-04-11 20:48:47] Started run 695 on task 146822, dataset 'segment'.
[2018-04-11 20:48:48] Error in run 688: Run already exists in server. Run id(s): {8960939}
[2018-04-11 20:48:48] Started run 696 on task 146822, dataset 'segment'.
[2018-04-11 20:48:49] Error in run 689: Run already exists in server. Run id(s): {8960941}
[2018-04-11 20:48:49] Started run 697 on task 146822, dataset 'segment'.
[2018-04-11 20:48:50] Error in run 690: Run already exists in server. Run id(s): {8960943}
[2018-04-11 20:48:50] Started run 698 on task 146822, dataset 'segment'.
[2018-04-11 20:48:50] Error in run 691: Run already exists in server. Run id(s): {8960945}
[2018-04-11 20:48:50] Started run 699 on task 146822, dataset 'segment'.
[2018-04-11 20:48:50] Error in run 693: Run already exists in server. Run id(s): {8960947}
[2018-04-11 20:48:50] Error in run 692: Run already exists in server. Run id(s): {8960944}
[2018-04-11 20:48:50] Started run 700 on task 146822, dataset 'segment'.
[2018-04-11 20:48:50] Started run 701 on task 146822, dataset 'segment'.
[2018-04-11 20:48:51] Error in run 694: Run already exists in server. Run id(s): {8960948}
[2018-04-11 20:48:51] Started run 702 on task 146822, dataset 'segment'.
[2018-04-11 20:48:51] Error in run 695: Run already exists in server. Run id(s): {8960950}
[2018-04-11 20:48:51] Started run 703 on task 146822, dataset 'segment'.
[2018-04-11 20:48:52] Error in run 696: Run already exists in server. Run id(s): {8960951}
[2018-04-11 20:48:52] Started run 704 on task 146822, dataset 'segment'.
[2018-04-11 20:48:54] Error in run 697: Run already exists in server. Run id(s): {8960953}
[2018-04-11 20:48:54] Started run 705 on task 146822, dataset 'segment'.
[2018-04-11 20:48:54] Error in run 698: Run already exists in server. Run id(s): {9015978, 8960954}
[2018-04-11 20:48:54] Started run 706 on task 146822, dataset 'segment'.
[2018-04-11 20:48:54] Error in run 699: Run already exists in server. Run id(s): {8960956}
[2018-04-11 20:48:54] Started run 707 on task 146822, dataset 'segment'.
[2018-04-11 20:48:55] Error in run 701: Run already exists in server. Run id(s): {8960959}
[2018-04-11 20:48:55] Started run 708 on task 146822, dataset 'segment'.
[2018-04-11 20:48:55] Error in run 700: Run already exists in server. Run id(s): {8960957}
[2018-04-11 20:48:55] Started run 709 on task 146822, dataset 'segment'.
[2018-04-11 20:48:55] Error in run 702: Run already exists in server. Run id(s): {8960960}
[2018-04-11 20:48:55] Started run 710 on task 146822, dataset 'segment'.
[2018-04-11 20:48:56] Error in run 703: Run already exists in server. Run id(s): {8960961}
[2018-04-11 20:48:56] Started run 711 on task 146822, dataset 'segment'.
[2018-04-11 20:48:57] Error in run 704: Run already exists in server. Run id(s): {8960963}
[2018-04-11 20:48:57] Started run 712 on task 146822, dataset 'segment'.
[2018-04-11 20:48:58] Error in run 705: Run already exists in server. Run id(s): {8960965}
[2018-04-11 20:48:58] Started run 713 on task 146822, dataset 'segment'.
[2018-04-11 20:48:59] Error in run 706: Run already exists in server. Run id(s): {8960967}
[2018-04-11 20:48:59] Error in run 707: Run already exists in server. Run id(s): {8960968}
[2018-04-11 20:48:59] Started run 714 on task 146822, dataset 'segment'.
[2018-04-11 20:48:59] Started run 715 on task 146822, dataset 'segment'.
[2018-04-11 20:48:59] Error in run 708: Run already exists in server. Run id(s): {8960970}
[2018-04-11 20:48:59] Started run 716 on task 146822, dataset 'segment'.
[2018-04-11 20:48:59] Error in run 709: Run already exists in server. Run id(s): {8960971}
[2018-04-11 20:48:59] Started run 717 on task 146822, dataset 'segment'.
[2018-04-11 20:49:00] Error in run 710: Run already exists in server. Run id(s): {8960972}
[2018-04-11 20:49:00] Started run 718 on task 146822, dataset 'segment'.
[2018-04-11 20:49:00] Error in run 711: Run already exists in server. Run id(s): {8960973}
[2018-04-11 20:49:00] Started run 719 on task 146822, dataset 'segment'.
[2018-04-11 20:49:01] Error in run 712: Run already exists in server. Run id(s): {8960976}
[2018-04-11 20:49:01] Started run 720 on task 146822, dataset 'segment'.
[2018-04-11 20:49:03] Error in run 713: Run already exists in server. Run id(s): {8960977}
[2018-04-11 20:49:03] Started run 721 on task 146822, dataset 'segment'.
[2018-04-11 20:49:03] Error in run 715: Run already exists in server. Run id(s): {8960979}
[2018-04-11 20:49:03] Started run 722 on task 146822, dataset 'segment'.
[2018-04-11 20:49:03] Error in run 714: Run already exists in server. Run id(s): {8960978}
[2018-04-11 20:49:03] Started run 723 on task 146822, dataset 'segment'.
[2018-04-11 20:49:04] Error in run 716: Run already exists in server. Run id(s): {8960982}
[2018-04-11 20:49:04] Started run 724 on task 146822, dataset 'segment'.
[2018-04-11 20:49:04] Error in run 717: Run already exists in server. Run id(s): {8960983}
[2018-04-11 20:49:04] Started run 725 on task 146822, dataset 'segment'.
[2018-04-11 20:49:04] Error in run 718: Run already exists in server. Run id(s): {8960985}
[2018-04-11 20:49:04] Started run 726 on task 146822, dataset 'segment'.
[2018-04-11 20:49:05] Error in run 719: Run already exists in server. Run id(s): {8960984}
[2018-04-11 20:49:05] Started run 727 on task 146822, dataset 'segment'.
[2018-04-11 20:49:06] Error in run 720: Run already exists in server. Run id(s): {8960987}
[2018-04-11 20:49:06] Started run 728 on task 146822, dataset 'segment'.
[2018-04-11 20:49:07] Error in run 721: Run already exists in server. Run id(s): {8960989}
[2018-04-11 20:49:07] Started run 729 on task 146822, dataset 'segment'.
[2018-04-11 20:49:08] Error in run 722: Run already exists in server. Run id(s): {8960991}
[2018-04-11 20:49:08] Error in run 724: Run already exists in server. Run id(s): {8960994}
[2018-04-11 20:49:08] Started run 730 on task 146822, dataset 'segment'.
[2018-04-11 20:49:08] Error in run 725: Run already exists in server. Run id(s): {8960995}
[2018-04-11 20:49:08] Error in run 723: Run already exists in server. Run id(s): {8960990}
[2018-04-11 20:49:08] Started run 731 on task 146822, dataset 'segment'.
[2018-04-11 20:49:08] Started run 732 on task 146822, dataset 'segment'.
[2018-04-11 20:49:08] Started run 733 on task 146822, dataset 'segment'.
[2018-04-11 20:49:09] Error in run 726: Run already exists in server. Run id(s): {8960996}
[2018-04-11 20:49:09] Started run 734 on task 146822, dataset 'segment'.
[2018-04-11 20:49:09] Error in run 727: Run already exists in server. Run id(s): {8960997}
[2018-04-11 20:49:09] Started run 735 on task 146822, dataset 'segment'.
[2018-04-11 20:49:10] Error in run 728: Run already exists in server. Run id(s): {8960999}
[2018-04-11 20:49:10] Started run 736 on task 146822, dataset 'segment'.
[2018-04-11 20:49:12] Error in run 729: Run already exists in server. Run id(s): {8961001}
[2018-04-11 20:49:12] Started run 737 on task 146822, dataset 'segment'.
[2018-04-11 20:49:13] Error in run 730: Run already exists in server. Run id(s): {8961004}
[2018-04-11 20:49:13] Started run 738 on task 146822, dataset 'segment'.
[2018-04-11 20:49:13] Error in run 731: Run already exists in server. Run id(s): {8961003}
[2018-04-11 20:49:13] Started run 739 on task 146822, dataset 'segment'.
[2018-04-11 20:49:13] Error in run 732: Run already exists in server. Run id(s): {8961005}
[2018-04-11 20:49:13] Started run 740 on task 146822, dataset 'segment'.
[2018-04-11 20:49:13] Error in run 733: Run already exists in server. Run id(s): {8961007}
[2018-04-11 20:49:13] Started run 741 on task 146822, dataset 'segment'.
[2018-04-11 20:49:14] Error in run 734: Run already exists in server. Run id(s): {8961009}
[2018-04-11 20:49:14] Started run 742 on task 146822, dataset 'segment'.
[2018-04-11 20:49:14] Error in run 735: Run already exists in server. Run id(s): {8961008}
[2018-04-11 20:49:14] Started run 743 on task 146822, dataset 'segment'.
[2018-04-11 20:49:15] Error in run 736: Run already exists in server. Run id(s): {8961011}
[2018-04-11 20:49:15] Started run 744 on task 146822, dataset 'segment'.
[2018-04-11 20:49:17] Error in run 737: Run already exists in server. Run id(s): {8961013}
[2018-04-11 20:49:17] Started run 745 on task 146822, dataset 'segment'.
[2018-04-11 20:49:17] Error in run 739: Run already exists in server. Run id(s): {8961015}
[2018-04-11 20:49:17] Started run 746 on task 146822, dataset 'segment'.
[2018-04-11 20:49:17] Error in run 740: Run already exists in server. Run id(s): {8961017}
[2018-04-11 20:49:17] Error in run 738: Run already exists in server. Run id(s): {8961016}
[2018-04-11 20:49:17] Started run 747 on task 146822, dataset 'segment'.
[2018-04-11 20:49:17] Started run 748 on task 146822, dataset 'segment'.
[2018-04-11 20:49:17] Error in run 741: Run already exists in server. Run id(s): {8961019}
[2018-04-11 20:49:17] Started run 749 on task 146822, dataset 'segment'.
[2018-04-11 20:49:18] Error in run 742: Run already exists in server. Run id(s): {8961020}
[2018-04-11 20:49:18] Started run 750 on task 146822, dataset 'segment'.
[2018-04-11 20:49:19] Error in run 743: Run already exists in server. Run id(s): {8961021}
[2018-04-11 20:49:19] Started run 751 on task 146822, dataset 'segment'.
[2018-04-11 20:49:19] Error in run 744: Run already exists in server. Run id(s): {8961023}
[2018-04-11 20:49:19] Started run 752 on task 146822, dataset 'segment'.
[2018-04-11 20:49:21] Error in run 745: Run already exists in server. Run id(s): {8961025}
[2018-04-11 20:49:21] Started run 753 on task 146822, dataset 'segment'.
[2018-04-11 20:49:22] Error in run 747: Run already exists in server. Run id(s): {8961028}
[2018-04-11 20:49:22] Error in run 746: Run already exists in server. Run id(s): {8961026}
[2018-04-11 20:49:22] Error in run 748: Run already exists in server. Run id(s): {8961029}
[2018-04-11 20:49:22] Started run 754 on task 146822, dataset 'segment'.
[2018-04-11 20:49:22] Started run 755 on task 146822, dataset 'segment'.
[2018-04-11 20:49:22] Started run 756 on task 146822, dataset 'segment'.
[2018-04-11 20:49:22] Error in run 749: Run already exists in server. Run id(s): {8961032}
[2018-04-11 20:49:22] Started run 757 on task 146822, dataset 'segment'.
[2018-04-11 20:49:23] Error in run 750: Run already exists in server. Run id(s): {8961031}
[2018-04-11 20:49:23] Started run 758 on task 146822, dataset 'segment'.
[2018-04-11 20:49:23] Error in run 751: Run already exists in server. Run id(s): {8961033}
[2018-04-11 20:49:23] Started run 759 on task 146822, dataset 'segment'.
[2018-04-11 20:49:24] Error in run 752: Run already exists in server. Run id(s): {8961035}
[2018-04-11 20:49:24] Started run 760 on task 146822, dataset 'segment'.
[2018-04-11 20:49:26] Error in run 753: Run already exists in server. Run id(s): {8961037}
[2018-04-11 20:49:26] Started run 761 on task 146822, dataset 'segment'.
[2018-04-11 20:49:26] Error in run 754: Run already exists in server. Run id(s): {8961038}
[2018-04-11 20:49:26] Error in run 755: Run already exists in server. Run id(s): {8961040}
[2018-04-11 20:49:26] Started run 762 on task 146822, dataset 'segment'.
[2018-04-11 20:49:26] Started run 763 on task 146822, dataset 'segment'.
[2018-04-11 20:49:26] Error in run 756: Run already exists in server. Run id(s): {8961041}
[2018-04-11 20:49:26] Started run 764 on task 146822, dataset 'segment'.
[2018-04-11 20:49:26] Error in run 757: Run already exists in server. Run id(s): {8961043}
[2018-04-11 20:49:26] Started run 765 on task 146822, dataset 'segment'.
[2018-04-11 20:49:27] Error in run 758: Run already exists in server. Run id(s): {8961044}
[2018-04-11 20:49:27] Started run 766 on task 146822, dataset 'segment'.
[2018-04-11 20:49:28] Error in run 759: Run already exists in server. Run id(s): {8961046}
[2018-04-11 20:49:28] Started run 767 on task 146822, dataset 'segment'.
[2018-04-11 20:49:29] Error in run 760: Run already exists in server. Run id(s): {8961047}
[2018-04-11 20:49:29] Started run 768 on task 146822, dataset 'segment'.
[2018-04-11 20:49:30] Error in run 761: Run already exists in server. Run id(s): {8961049}
[2018-04-11 20:49:30] Started run 769 on task 146822, dataset 'segment'.
[2018-04-11 20:49:31] Error in run 763: Run already exists in server. Run id(s): {8961052}
[2018-04-11 20:49:31] Started run 770 on task 146822, dataset 'segment'.
[2018-04-11 20:49:31] Error in run 764: Run already exists in server. Run id(s): {8961054}
[2018-04-11 20:49:31] Error in run 762: Run already exists in server. Run id(s): {8961050}
[2018-04-11 20:49:31] Started run 771 on task 146822, dataset 'segment'.
[2018-04-11 20:49:31] Started run 772 on task 146822, dataset 'segment'.
[2018-04-11 20:49:31] Error in run 765: Run already exists in server. Run id(s): {8961055}
[2018-04-11 20:49:31] Started run 773 on task 146822, dataset 'segment'.
[2018-04-11 20:49:32] Error in run 766: Run already exists in server. Run id(s): {8961056}
[2018-04-11 20:49:32] Started run 774 on task 146822, dataset 'segment'.
[2018-04-11 20:49:32] Error in run 767: Run already exists in server. Run id(s): {8961057}
[2018-04-11 20:49:32] Started run 775 on task 146822, dataset 'segment'.
[2018-04-11 20:49:33] Error in run 768: Run already exists in server. Run id(s): {8961059}
[2018-04-11 20:49:33] Started run 776 on task 146822, dataset 'segment'.
[2018-04-11 20:49:35] Error in run 769: Run already exists in server. Run id(s): {8961061}
[2018-04-11 20:49:35] Started run 777 on task 146822, dataset 'segment'.
[2018-04-11 20:49:35] Error in run 771: Run already exists in server. Run id(s): {8961063}
[2018-04-11 20:49:35] Started run 778 on task 146822, dataset 'segment'.
[2018-04-11 20:49:35] Error in run 770: Run already exists in server. Run id(s): {8961064}
[2018-04-11 20:49:35] Started run 779 on task 146822, dataset 'segment'.
[2018-04-11 20:49:35] Error in run 772: Run already exists in server. Run id(s): {8961066}
[2018-04-11 20:49:35] Started run 780 on task 146822, dataset 'segment'.
[2018-04-11 20:49:36] Error in run 773: Run already exists in server. Run id(s): {8961068}
[2018-04-11 20:49:36] Started run 781 on task 146822, dataset 'segment'.
[2018-04-11 20:49:36] Error in run 774: Run already exists in server. Run id(s): {8961069}
[2018-04-11 20:49:36] Started run 782 on task 146822, dataset 'segment'.
[2018-04-11 20:49:37] Error in run 775: Run already exists in server. Run id(s): {8961067}
[2018-04-11 20:49:37] Started run 783 on task 146822, dataset 'segment'.
[2018-04-11 20:49:38] Error in run 776: Run already exists in server. Run id(s): {8961071}
[2018-04-11 20:49:38] Started run 784 on task 146822, dataset 'segment'.
[2018-04-11 20:49:39] Error in run 777: Run already exists in server. Run id(s): {8961074}
[2018-04-11 20:49:39] Started run 785 on task 146822, dataset 'segment'.
[2018-04-11 20:49:40] Error in run 779: Run already exists in server. Run id(s): {8961075}
[2018-04-11 20:49:40] Started run 786 on task 146822, dataset 'segment'.
[2018-04-11 20:49:40] Error in run 778: Run already exists in server. Run id(s): {8961076}
[2018-04-11 20:49:40] Started run 787 on task 146822, dataset 'segment'.
[2018-04-11 20:49:40] Error in run 780: Run already exists in server. Run id(s): {8961077}
[2018-04-11 20:49:40] Started run 788 on task 146822, dataset 'segment'.
[2018-04-11 20:49:40] Error in run 781: Run already exists in server. Run id(s): {8961080}
[2018-04-11 20:49:40] Started run 789 on task 146822, dataset 'segment'.
[2018-04-11 20:49:41] Error in run 782: Run already exists in server. Run id(s): {8961079}
[2018-04-11 20:49:41] Started run 790 on task 146822, dataset 'segment'.
[2018-04-11 20:49:41] Error in run 783: Run already exists in server. Run id(s): {8961082}
[2018-04-11 20:49:41] Started run 791 on task 146822, dataset 'segment'.
[2018-04-11 20:49:42] Error in run 784: Run already exists in server. Run id(s): {8961083}
[2018-04-11 20:49:42] Started run 792 on task 146822, dataset 'segment'.
[2018-04-11 20:49:44] Error in run 785: Run already exists in server. Run id(s): {8961084}
[2018-04-11 20:49:44] Started run 793 on task 146822, dataset 'segment'.
[2018-04-11 20:49:44] Error in run 786: Run already exists in server. Run id(s): {8961086}
[2018-04-11 20:49:44] Started run 794 on task 146822, dataset 'segment'.
[2018-04-11 20:49:44] Error in run 787: Run already exists in server. Run id(s): {8961087}
[2018-04-11 20:49:44] Started run 795 on task 146822, dataset 'segment'.
[2018-04-11 20:49:44] Error in run 788: Run already exists in server. Run id(s): {8961089}
[2018-04-11 20:49:44] Started run 796 on task 146822, dataset 'segment'.
[2018-04-11 20:49:45] Error in run 789: Run already exists in server. Run id(s): {8961091}
[2018-04-11 20:49:45] Started run 797 on task 146822, dataset 'segment'.
[2018-04-11 20:49:45] Error in run 790: Run already exists in server. Run id(s): {8961092}
[2018-04-11 20:49:45] Started run 798 on task 146822, dataset 'segment'.
[2018-04-11 20:49:46] Error in run 791: Run already exists in server. Run id(s): {8961093}
[2018-04-11 20:49:46] Started run 799 on task 146822, dataset 'segment'.
[2018-04-11 20:49:47] Error in run 792: Run already exists in server. Run id(s): {8961095}
[2018-04-11 20:49:47] Started run 800 on task 146822, dataset 'segment'.
[2018-04-11 20:49:49] Error in run 795: Run already exists in server. Run id(s): {8961097}
[2018-04-11 20:49:49] Started run 801 on task 146822, dataset 'segment'.
[2018-04-11 20:49:49] Error in run 794: Run already exists in server. Run id(s): {8961099}
[2018-04-11 20:49:49] Error in run 793: Run already exists in server. Run id(s): {8961096}
[2018-04-11 20:49:49] Started run 802 on task 146822, dataset 'segment'.
[2018-04-11 20:49:49] Started run 803 on task 146822, dataset 'segment'.
[2018-04-11 20:49:49] Error in run 796: Run already exists in server. Run id(s): {8961102}
[2018-04-11 20:49:49] Started run 804 on task 146822, dataset 'segment'.
[2018-04-11 20:49:49] Error in run 797: Run already exists in server. Run id(s): {8961103}
[2018-04-11 20:49:49] Started run 805 on task 146822, dataset 'segment'.
[2018-04-11 20:49:50] Error in run 798: Run already exists in server. Run id(s): {8961104}
[2018-04-11 20:49:50] Started run 806 on task 146822, dataset 'segment'.
[2018-04-11 20:49:50] Error in run 799: Run already exists in server. Run id(s): {8961105}
[2018-04-11 20:49:50] Started run 807 on task 146822, dataset 'segment'.
[2018-04-11 20:49:51] Error in run 800: Run already exists in server. Run id(s): {8961107}
[2018-04-11 20:49:51] Started run 808 on task 146822, dataset 'segment'.
[2018-04-11 20:49:53] Error in run 802: Run already exists in server. Run id(s): {8961110}
[2018-04-11 20:49:53] Error in run 803: Run already exists in server. Run id(s): {8961111}
[2018-04-11 20:49:53] Started run 809 on task 146822, dataset 'segment'.
[2018-04-11 20:49:53] Started run 810 on task 146822, dataset 'segment'.
[2018-04-11 20:49:53] Error in run 801: Run already exists in server. Run id(s): {8961108}
[2018-04-11 20:49:53] Started run 811 on task 146822, dataset 'segment'.
[2018-04-11 20:49:54] Error in run 804: Run already exists in server. Run id(s): {8961113}
[2018-04-11 20:49:54] Started run 812 on task 146822, dataset 'segment'.
[2018-04-11 20:49:54] Error in run 805: Run already exists in server. Run id(s): {8961115}
[2018-04-11 20:49:54] Started run 813 on task 146822, dataset 'segment'.
[2018-04-11 20:49:54] Error in run 806: Run already exists in server. Run id(s): {8961117}
[2018-04-11 20:49:54] Started run 814 on task 146822, dataset 'segment'.
[2018-04-11 20:49:55] Error in run 807: Run already exists in server. Run id(s): {8961118}
[2018-04-11 20:49:55] Started run 815 on task 146822, dataset 'segment'.
[2018-04-11 20:49:56] Error in run 808: Run already exists in server. Run id(s): {8961119}
[2018-04-11 20:49:56] Started run 816 on task 146822, dataset 'segment'.
[2018-04-11 20:49:58] Error in run 809: Run already exists in server. Run id(s): {8961120}
[2018-04-11 20:49:58] Error in run 810: Run already exists in server. Run id(s): {8961121}
[2018-04-11 20:49:58] Started run 817 on task 146822, dataset 'segment'.
[2018-04-11 20:49:58] Started run 818 on task 146822, dataset 'segment'.
[2018-04-11 20:49:58] Error in run 811: Run already exists in server. Run id(s): {8961123}
[2018-04-11 20:49:58] Started run 819 on task 146822, dataset 'segment'.
[2018-04-11 20:49:58] Error in run 812: Run already exists in server. Run id(s): {8961124}
[2018-04-11 20:49:58] Started run 820 on task 146822, dataset 'segment'.
[2018-04-11 20:49:58] Error in run 813: Run already exists in server. Run id(s): {8961128}
[2018-04-11 20:49:58] Started run 821 on task 146822, dataset 'segment'.
[2018-04-11 20:49:59] Error in run 814: Run already exists in server. Run id(s): {8961127}
[2018-04-11 20:49:59] Started run 822 on task 146822, dataset 'segment'.
[2018-04-11 20:49:59] Error in run 815: Run already exists in server. Run id(s): {8961129}
[2018-04-11 20:49:59] Started run 823 on task 146822, dataset 'segment'.
[2018-04-11 20:50:00] Error in run 816: Run already exists in server. Run id(s): {8961131}
[2018-04-11 20:50:00] Started run 824 on task 146822, dataset 'segment'.
[2018-04-11 20:50:02] Error in run 817: Run already exists in server. Run id(s): {8961132}
[2018-04-11 20:50:02] Started run 825 on task 146822, dataset 'segment'.
[2018-04-11 20:50:02] Error in run 818: Run already exists in server. Run id(s): {8961133}
[2018-04-11 20:50:02] Started run 826 on task 146822, dataset 'segment'.
[2018-04-11 20:50:02] Error in run 819: Run already exists in server. Run id(s): {8961134}
[2018-04-11 20:50:02] Started run 827 on task 146822, dataset 'segment'.
[2018-04-11 20:50:03] Error in run 820: Run already exists in server. Run id(s): {8961136}
[2018-04-11 20:50:03] Error in run 821: Run already exists in server. Run id(s): {8961139}
[2018-04-11 20:50:03] Started run 828 on task 146822, dataset 'segment'.
[2018-04-11 20:50:03] Started run 829 on task 146822, dataset 'segment'.
[2018-04-11 20:50:03] Error in run 822: Run already exists in server. Run id(s): {8961140}
[2018-04-11 20:50:03] Started run 830 on task 146822, dataset 'segment'.
[2018-04-11 20:50:04] Error in run 823: Run already exists in server. Run id(s): {8961141}
[2018-04-11 20:50:04] Started run 831 on task 146822, dataset 'segment'.
[2018-04-11 20:50:05] Error in run 824: Run already exists in server. Run id(s): {8961143}
[2018-04-11 20:50:05] Started run 832 on task 146822, dataset 'segment'.
[2018-04-11 20:50:07] Error in run 826: Run already exists in server. Run id(s): {8961144}
[2018-04-11 20:50:07] Started run 833 on task 146822, dataset 'segment'.
[2018-04-11 20:50:07] Error in run 825: Run already exists in server. Run id(s): {8961145}
[2018-04-11 20:50:07] Started run 834 on task 146822, dataset 'segment'.
[2018-04-11 20:50:07] Error in run 827: Run already exists in server. Run id(s): {8961146}
[2018-04-11 20:50:07] Started run 835 on task 146822, dataset 'segment'.
[2018-04-11 20:50:07] Error in run 829: Run already exists in server. Run id(s): {8961151}
[2018-04-11 20:50:07] Started run 836 on task 146822, dataset 'segment'.
[2018-04-11 20:50:07] Error in run 828: Run already exists in server. Run id(s): {8961148}
[2018-04-11 20:50:07] Started run 837 on task 146822, dataset 'segment'.
[2018-04-11 20:50:08] Error in run 830: Run already exists in server. Run id(s): {8961153}
[2018-04-11 20:50:08] Started run 838 on task 146822, dataset 'segment'.
[2018-04-11 20:50:08] Error in run 831: Run already exists in server. Run id(s): {8961152}
[2018-04-11 20:50:08] Started run 839 on task 146822, dataset 'segment'.
[2018-04-11 20:50:09] Error in run 832: Run already exists in server. Run id(s): {8961155}
[2018-04-11 20:50:09] Started run 840 on task 146822, dataset 'segment'.
[2018-04-11 20:50:11] Error in run 833: Run already exists in server. Run id(s): {8961156}
[2018-04-11 20:50:11] Error in run 834: Run already exists in server. Run id(s): {8961158}
[2018-04-11 20:50:11] Error in run 835: Run already exists in server. Run id(s): {8961157}
[2018-04-11 20:50:11] Started run 841 on task 146822, dataset 'segment'.
[2018-04-11 20:50:11] Started run 842 on task 146822, dataset 'segment'.
[2018-04-11 20:50:11] Started run 843 on task 146822, dataset 'segment'.
[2018-04-11 20:50:12] Error in run 837: Run already exists in server. Run id(s): {8961163}
[2018-04-11 20:50:12] Started run 844 on task 146822, dataset 'segment'.
[2018-04-11 20:50:12] Error in run 836: Run already exists in server. Run id(s): {8961159}
[2018-04-11 20:50:12] Started run 845 on task 146822, dataset 'segment'.
[2018-04-11 20:50:12] Error in run 838: Run already exists in server. Run id(s): {8961164}
[2018-04-11 20:50:12] Started run 846 on task 146822, dataset 'segment'.
[2018-04-11 20:50:13] Error in run 839: Run already exists in server. Run id(s): {8961165}
[2018-04-11 20:50:13] Started run 847 on task 146822, dataset 'segment'.
[2018-04-11 20:50:14] Error in run 840: Run already exists in server. Run id(s): {8961166}
[2018-04-11 20:50:14] Started run 848 on task 146822, dataset 'segment'.
[2018-04-11 20:50:16] Error in run 841: Run already exists in server. Run id(s): {8961168}
[2018-04-11 20:50:16] Started run 849 on task 146822, dataset 'segment'.
[2018-04-11 20:50:16] Error in run 842: Run already exists in server. Run id(s): {8961169}
[2018-04-11 20:50:16] Error in run 843: Run already exists in server. Run id(s): {8961170}
[2018-04-11 20:50:16] Started run 850 on task 146822, dataset 'segment'.
[2018-04-11 20:50:16] Error in run 844: Run already exists in server. Run id(s): {8961171}
[2018-04-11 20:50:16] Started run 851 on task 146822, dataset 'segment'.
[2018-04-11 20:50:16] Started run 852 on task 146822, dataset 'segment'.
[2018-04-11 20:50:16] Error in run 845: Run already exists in server. Run id(s): {8961174}
[2018-04-11 20:50:16] Started run 853 on task 146822, dataset 'segment'.
[2018-04-11 20:50:17] Error in run 846: Run already exists in server. Run id(s): {8961175}
[2018-04-11 20:50:17] Started run 854 on task 146822, dataset 'segment'.
[2018-04-11 20:50:17] Error in run 847: Run already exists in server. Run id(s): {8961177}
[2018-04-11 20:50:17] Started run 855 on task 146822, dataset 'segment'.
[2018-04-11 20:50:18] Error in run 848: Run already exists in server. Run id(s): {8961178}
[2018-04-11 20:50:18] Started run 856 on task 146822, dataset 'segment'.
[2018-04-11 20:50:21] Error in run 849: Run already exists in server. Run id(s): {8961180}
[2018-04-11 20:50:21] Started run 857 on task 146822, dataset 'segment'.
[2018-04-11 20:50:21] Error in run 850: Run already exists in server. Run id(s): {8961181}
[2018-04-11 20:50:21] Started run 858 on task 146822, dataset 'segment'.
[2018-04-11 20:50:21] Error in run 851: Run already exists in server. Run id(s): {8961183}
[2018-04-11 20:50:21] Started run 859 on task 146822, dataset 'segment'.
[2018-04-11 20:50:21] Error in run 853: Run already exists in server. Run id(s): {8961185}
[2018-04-11 20:50:21] Started run 860 on task 146822, dataset 'segment'.
[2018-04-11 20:50:21] Error in run 852: Run already exists in server. Run id(s): {8961182}
[2018-04-11 20:50:21] Started run 861 on task 146822, dataset 'segment'.
[2018-04-11 20:50:22] Error in run 854: Run already exists in server. Run id(s): {8961187}
[2018-04-11 20:50:22] Started run 862 on task 146822, dataset 'segment'.
[2018-04-11 20:50:22] Error in run 855: Run already exists in server. Run id(s): {8961189}
[2018-04-11 20:50:22] Started run 863 on task 146822, dataset 'segment'.
[2018-04-11 20:50:23] Error in run 856: Run already exists in server. Run id(s): {8961190}
[2018-04-11 20:50:23] Started run 864 on task 146822, dataset 'segment'.
[2018-04-11 20:50:25] Error in run 857: Run already exists in server. Run id(s): {8961192}
[2018-04-11 20:50:25] Started run 865 on task 146822, dataset 'segment'.
[2018-04-11 20:50:25] Error in run 858: Run already exists in server. Run id(s): {8961193}
[2018-04-11 20:50:25] Error in run 859: Run already exists in server. Run id(s): {8961194}
[2018-04-11 20:50:25] Started run 866 on task 146822, dataset 'segment'.
[2018-04-11 20:50:25] Started run 867 on task 146822, dataset 'segment'.
[2018-04-11 20:50:25] Error in run 860: Run already exists in server. Run id(s): {8961195}
[2018-04-11 20:50:25] Error in run 861: Run already exists in server. Run id(s): {8961196}
[2018-04-11 20:50:25] Started run 868 on task 146822, dataset 'segment'.
[2018-04-11 20:50:25] Started run 869 on task 146822, dataset 'segment'.
[2018-04-11 20:50:26] Error in run 862: Run already exists in server. Run id(s): {8961200}
[2018-04-11 20:50:26] Started run 870 on task 146822, dataset 'segment'.
[2018-04-11 20:50:27] Error in run 863: Run already exists in server. Run id(s): {8961198}
[2018-04-11 20:50:27] Started run 871 on task 146822, dataset 'segment'.
[2018-04-11 20:50:27] Error in run 864: Run already exists in server. Run id(s): {8961202}
[2018-04-11 20:50:27] Started run 872 on task 146822, dataset 'segment'.
[2018-04-11 20:50:30] Error in run 865: Run already exists in server. Run id(s): {8961203}
[2018-04-11 20:50:30] Started run 873 on task 146822, dataset 'segment'.
[2018-04-11 20:50:30] Error in run 866: Run already exists in server. Run id(s): {8961205}
[2018-04-11 20:50:30] Error in run 867: Run already exists in server. Run id(s): {8961206}
[2018-04-11 20:50:30] Started run 874 on task 146822, dataset 'segment'.
[2018-04-11 20:50:30] Started run 875 on task 146822, dataset 'segment'.
[2018-04-11 20:50:30] Error in run 868: Run already exists in server. Run id(s): {8961207}
[2018-04-11 20:50:30] Started run 876 on task 146822, dataset 'segment'.
[2018-04-11 20:50:30] Error in run 869: Run already exists in server. Run id(s): {8961208}
[2018-04-11 20:50:30] Started run 877 on task 146822, dataset 'segment'.
[2018-04-11 20:50:31] Error in run 870: Run already exists in server. Run id(s): {8961210}
[2018-04-11 20:50:31] Started run 878 on task 146822, dataset 'segment'.
[2018-04-11 20:50:31] Error in run 871: Run already exists in server. Run id(s): {8961211}
[2018-04-11 20:50:31] Started run 879 on task 146822, dataset 'segment'.
[2018-04-11 20:50:32] Error in run 872: Run already exists in server. Run id(s): {8961214}
[2018-04-11 20:50:32] Started run 880 on task 146822, dataset 'segment'.
[2018-04-11 20:50:34] Error in run 873: Run already exists in server. Run id(s): {8961216}
[2018-04-11 20:50:34] Started run 881 on task 146822, dataset 'segment'.
[2018-04-11 20:50:34] Error in run 874: Run already exists in server. Run id(s): {8961217}
[2018-04-11 20:50:34] Started run 882 on task 146822, dataset 'segment'.
[2018-04-11 20:50:34] Error in run 875: Run already exists in server. Run id(s): {8961218}
[2018-04-11 20:50:34] Started run 883 on task 146822, dataset 'segment'.
[2018-04-11 20:50:34] Error in run 876: Run already exists in server. Run id(s): {8961219}
[2018-04-11 20:50:34] Started run 884 on task 146822, dataset 'segment'.
[2018-04-11 20:50:34] Error in run 877: Run already exists in server. Run id(s): {8961220}
[2018-04-11 20:50:34] Started run 885 on task 146822, dataset 'segment'.
[2018-04-11 20:50:35] Error in run 878: Run already exists in server. Run id(s): {8961221}
[2018-04-11 20:50:35] Started run 886 on task 146822, dataset 'segment'.
[2018-04-11 20:50:36] Error in run 879: Run already exists in server. Run id(s): {8961222}
[2018-04-11 20:50:36] Started run 887 on task 146822, dataset 'segment'.
[2018-04-11 20:50:36] Error in run 880: Run already exists in server. Run id(s): {8961225}
[2018-04-11 20:50:36] Started run 888 on task 146822, dataset 'segment'.
[2018-04-11 20:50:39] Error in run 881: Run already exists in server. Run id(s): {8961228}
[2018-04-11 20:50:39] Error in run 882: Run already exists in server. Run id(s): {8961229}
[2018-04-11 20:50:39] Started run 889 on task 146822, dataset 'segment'.
[2018-04-11 20:50:39] Error in run 883: Run already exists in server. Run id(s): {8961230}
[2018-04-11 20:50:39] Started run 890 on task 146822, dataset 'segment'.
[2018-04-11 20:50:39] Started run 891 on task 146822, dataset 'segment'.
[2018-04-11 20:50:39] Error in run 884: Run already exists in server. Run id(s): {8961231}
[2018-04-11 20:50:39] Started run 892 on task 146822, dataset 'segment'.
[2018-04-11 20:50:39] Error in run 885: Run already exists in server. Run id(s): {8961232}
[2018-04-11 20:50:39] Started run 893 on task 146822, dataset 'segment'.
[2018-04-11 20:50:40] Error in run 886: Run already exists in server. Run id(s): {8961233}
[2018-04-11 20:50:40] Started run 894 on task 146822, dataset 'segment'.
[2018-04-11 20:50:40] Error in run 887: Run already exists in server. Run id(s): {8961234}
[2018-04-11 20:50:40] Started run 895 on task 146822, dataset 'segment'.
[2018-04-11 20:50:41] Error in run 888: Run already exists in server. Run id(s): {8961237}
[2018-04-11 20:50:41] Started run 896 on task 146822, dataset 'segment'.
[2018-04-11 20:50:43] Error in run 889: Run already exists in server. Run id(s): {8961240}
[2018-04-11 20:50:43] Error in run 890: Run already exists in server. Run id(s): {8961242}
[2018-04-11 20:50:43] Started run 897 on task 146822, dataset 'segment'.
[2018-04-11 20:50:43] Started run 898 on task 146822, dataset 'segment'.
[2018-04-11 20:50:43] Error in run 892: Run already exists in server. Run id(s): {8961243}
[2018-04-11 20:50:43] Error in run 891: Run already exists in server. Run id(s): {8961241}
[2018-04-11 20:50:43] Started run 899 on task 146822, dataset 'segment'.
[2018-04-11 20:50:43] Started run 900 on task 146822, dataset 'segment'.
[2018-04-11 20:50:44] Error in run 893: Run already exists in server. Run id(s): {8961244}
[2018-04-11 20:50:44] Started run 901 on task 146822, dataset 'segment'.
[2018-04-11 20:50:44] Error in run 894: Run already exists in server. Run id(s): {8961245}
[2018-04-11 20:50:44] Started run 902 on task 146822, dataset 'segment'.
[2018-04-11 20:50:45] Error in run 895: Run already exists in server. Run id(s): {8961247}
[2018-04-11 20:50:45] Started run 903 on task 146822, dataset 'segment'.
[2018-04-11 20:50:46] Error in run 896: Run already exists in server. Run id(s): {8961249}
[2018-04-11 20:50:46] Started run 904 on task 146822, dataset 'segment'.
[2018-04-11 20:50:48] Error in run 898: Run already exists in server. Run id(s): {8961253}
[2018-04-11 20:50:48] Started run 905 on task 146822, dataset 'segment'.
[2018-04-11 20:50:48] Error in run 897: Run already exists in server. Run id(s): {8961251}
[2018-04-11 20:50:48] Started run 906 on task 146822, dataset 'segment'.
[2018-04-11 20:50:48] Error in run 899: Run already exists in server. Run id(s): {8961254}
[2018-04-11 20:50:48] Started run 907 on task 146822, dataset 'segment'.
[2018-04-11 20:50:48] Error in run 900: Run already exists in server. Run id(s): {8961255}
[2018-04-11 20:50:48] Started run 908 on task 146822, dataset 'segment'.
[2018-04-11 20:50:48] Error in run 901: Run already exists in server. Run id(s): {8961256}
[2018-04-11 20:50:48] Started run 909 on task 146822, dataset 'segment'.
[2018-04-11 20:50:49] Error in run 902: Run already exists in server. Run id(s): {8961257}
[2018-04-11 20:50:49] Started run 910 on task 146822, dataset 'segment'.
[2018-04-11 20:50:50] Error in run 903: Run already exists in server. Run id(s): {8961258}
[2018-04-11 20:50:50] Started run 911 on task 146822, dataset 'segment'.
[2018-04-11 20:50:50] Error in run 904: Run already exists in server. Run id(s): {8961260}
[2018-04-11 20:50:50] Started run 912 on task 146822, dataset 'segment'.
[2018-04-11 20:50:52] Error in run 905: Run already exists in server. Run id(s): {8961263}
[2018-04-11 20:50:52] Started run 913 on task 146822, dataset 'segment'.
[2018-04-11 20:50:52] Error in run 906: Run already exists in server. Run id(s): {8961266}
[2018-04-11 20:50:52] Started run 914 on task 146822, dataset 'segment'.
[2018-04-11 20:50:53] Error in run 907: Run already exists in server. Run id(s): {8961264}
[2018-04-11 20:50:53] Started run 915 on task 146822, dataset 'segment'.
[2018-04-11 20:50:53] Error in run 908: Run already exists in server. Run id(s): {8961267}
[2018-04-11 20:50:53] Started run 916 on task 146822, dataset 'segment'.
[2018-04-11 20:50:53] Error in run 909: Run already exists in server. Run id(s): {8961268}
[2018-04-11 20:50:53] Started run 917 on task 146822, dataset 'segment'.
[2018-04-11 20:50:54] Error in run 910: Run already exists in server. Run id(s): {8961269}
[2018-04-11 20:50:54] Started run 918 on task 146822, dataset 'segment'.
[2018-04-11 20:50:54] Error in run 911: Run already exists in server. Run id(s): {8961270}
[2018-04-11 20:50:54] Started run 919 on task 146822, dataset 'segment'.
[2018-04-11 20:50:55] Error in run 912: Run already exists in server. Run id(s): {8961272}
[2018-04-11 20:50:55] Started run 920 on task 146822, dataset 'segment'.
[2018-04-11 20:50:57] Error in run 913: Run already exists in server. Run id(s): {8961275}
[2018-04-11 20:50:57] Started run 921 on task 146822, dataset 'segment'.
[2018-04-11 20:50:57] Error in run 914: Run already exists in server. Run id(s): {8961274}
[2018-04-11 20:50:57] Started run 922 on task 146822, dataset 'segment'.
[2018-04-11 20:50:57] Error in run 916: Run already exists in server. Run id(s): {8961279}
[2018-04-11 20:50:57] Error in run 915: Run already exists in server. Run id(s): {8961278}
[2018-04-11 20:50:57] Started run 923 on task 146822, dataset 'segment'.
[2018-04-11 20:50:57] Started run 924 on task 146822, dataset 'segment'.
[2018-04-11 20:50:57] Error in run 917: Run already exists in server. Run id(s): {8961280}
[2018-04-11 20:50:57] Started run 925 on task 146822, dataset 'segment'.
[2018-04-11 20:50:58] Error in run 918: Run already exists in server. Run id(s): {8961281}
[2018-04-11 20:50:58] Started run 926 on task 146822, dataset 'segment'.
[2018-04-11 20:50:59] Error in run 919: Run already exists in server. Run id(s): {8961282}
[2018-04-11 20:50:59] Started run 927 on task 146822, dataset 'segment'.
[2018-04-11 20:50:59] Error in run 920: Run already exists in server. Run id(s): {8961284}
[2018-04-11 20:50:59] Started run 928 on task 146822, dataset 'segment'.
[2018-04-11 20:51:01] Error in run 921: Run already exists in server. Run id(s): {8961287}
[2018-04-11 20:51:01] Started run 929 on task 146822, dataset 'segment'.
[2018-04-11 20:51:01] Error in run 922: Run already exists in server. Run id(s): {8961288}
[2018-04-11 20:51:01] Started run 930 on task 146822, dataset 'segment'.
[2018-04-11 20:51:02] Error in run 924: Run already exists in server. Run id(s): {8961291}
[2018-04-11 20:51:02] Started run 931 on task 146822, dataset 'segment'.
[2018-04-11 20:51:02] Error in run 923: Run already exists in server. Run id(s): {8961290}
[2018-04-11 20:51:02] Started run 932 on task 146822, dataset 'segment'.
[2018-04-11 20:51:02] Error in run 925: Run already exists in server. Run id(s): {8961292}
[2018-04-11 20:51:02] Started run 933 on task 146822, dataset 'segment'.
[2018-04-11 20:51:03] Error in run 926: Run already exists in server. Run id(s): {8961293}
[2018-04-11 20:51:03] Started run 934 on task 146822, dataset 'segment'.
[2018-04-11 20:51:03] Error in run 927: Run already exists in server. Run id(s): {8961294}
[2018-04-11 20:51:03] Started run 935 on task 146822, dataset 'segment'.
[2018-04-11 20:51:04] Error in run 928: Run already exists in server. Run id(s): {8961296}
[2018-04-11 20:51:04] Started run 936 on task 146822, dataset 'segment'.
[2018-04-11 20:51:06] Error in run 929: Run already exists in server. Run id(s): {8961298}
[2018-04-11 20:51:06] Started run 937 on task 146822, dataset 'segment'.
[2018-04-11 20:51:06] Error in run 931: Run already exists in server. Run id(s): {8961300}
[2018-04-11 20:51:06] Started run 938 on task 146822, dataset 'segment'.
[2018-04-11 20:51:06] Error in run 932: Run already exists in server. Run id(s): {8961302}
[2018-04-11 20:51:06] Started run 939 on task 146822, dataset 'segment'.
[2018-04-11 20:51:06] Error in run 930: Run already exists in server. Run id(s): {8961299}
[2018-04-11 20:51:06] Started run 940 on task 146822, dataset 'segment'.
[2018-04-11 20:51:07] Error in run 933: Run already exists in server. Run id(s): {8961305}
[2018-04-11 20:51:07] Started run 941 on task 146822, dataset 'segment'.
[2018-04-11 20:51:07] Error in run 934: Run already exists in server. Run id(s): {8961306}
[2018-04-11 20:51:07] Started run 942 on task 146822, dataset 'segment'.
[2018-04-11 20:51:08] Error in run 935: Run already exists in server. Run id(s): {8961307}
[2018-04-11 20:51:08] Started run 943 on task 146822, dataset 'segment'.
[2018-04-11 20:51:08] Error in run 936: Run already exists in server. Run id(s): {8961308}
[2018-04-11 20:51:08] Started run 944 on task 146822, dataset 'segment'.
[2018-04-11 20:51:10] Error in run 937: Run already exists in server. Run id(s): {8961310}
[2018-04-11 20:51:10] Started run 945 on task 146822, dataset 'segment'.
[2018-04-11 20:51:11] Error in run 938: Run already exists in server. Run id(s): {8961311}
[2018-04-11 20:51:11] Started run 946 on task 146822, dataset 'segment'.
[2018-04-11 20:51:11] Error in run 939: Run already exists in server. Run id(s): {8961312}
[2018-04-11 20:51:11] Error in run 940: Run already exists in server. Run id(s): {8961314}
[2018-04-11 20:51:11] Started run 947 on task 146822, dataset 'segment'.
[2018-04-11 20:51:11] Started run 948 on task 146822, dataset 'segment'.
[2018-04-11 20:51:11] Error in run 941: Run already exists in server. Run id(s): {8961317}
[2018-04-11 20:51:11] Started run 949 on task 146822, dataset 'segment'.
[2018-04-11 20:51:12] Error in run 942: Run already exists in server. Run id(s): {8961318}
[2018-04-11 20:51:12] Started run 950 on task 146822, dataset 'segment'.
[2018-04-11 20:51:12] Error in run 943: Run already exists in server. Run id(s): {8961319}
[2018-04-11 20:51:12] Started run 951 on task 146822, dataset 'segment'.
[2018-04-11 20:51:13] Error in run 944: Run already exists in server. Run id(s): {8961320}
[2018-04-11 20:51:13] Started run 952 on task 146822, dataset 'segment'.
[2018-04-11 20:51:15] Error in run 945: Run already exists in server. Run id(s): {8961322}
[2018-04-11 20:51:15] Started run 953 on task 146822, dataset 'segment'.
[2018-04-11 20:51:15] Error in run 948: Run already exists in server. Run id(s): {8961325}
[2018-04-11 20:51:15] Started run 954 on task 146822, dataset 'segment'.
[2018-04-11 20:51:15] Error in run 949: Run already exists in server. Run id(s): {8961328}
[2018-04-11 20:51:15] Started run 955 on task 146822, dataset 'segment'.
[2018-04-11 20:51:16] Error in run 947: Run already exists in server. Run id(s): {8961324}
[2018-04-11 20:51:16] Started run 956 on task 146822, dataset 'segment'.
[2018-04-11 20:51:16] Error in run 946: Run already exists in server. Run id(s): {8961323}
[2018-04-11 20:51:16] Started run 957 on task 146822, dataset 'segment'.
[2018-04-11 20:51:16] Error in run 950: Run already exists in server. Run id(s): {8961331}
[2018-04-11 20:51:16] Started run 958 on task 146822, dataset 'segment'.
[2018-04-11 20:51:17] Error in run 951: Run already exists in server. Run id(s): {8961329}
[2018-04-11 20:51:17] Started run 959 on task 146822, dataset 'segment'.
[2018-04-11 20:51:18] Error in run 952: Run already exists in server. Run id(s): {8961332}
[2018-04-11 20:51:18] Started run 960 on task 146822, dataset 'segment'.
[2018-04-11 20:51:19] Error in run 953: Run already exists in server. Run id(s): {8961333}
[2018-04-11 20:51:19] Started run 961 on task 146822, dataset 'segment'.
[2018-04-11 20:51:20] Error in run 954: Run already exists in server. Run id(s): {8961335}
[2018-04-11 20:51:20] Started run 962 on task 146822, dataset 'segment'.
[2018-04-11 20:51:20] Error in run 955: Run already exists in server. Run id(s): {8961336}
[2018-04-11 20:51:20] Started run 963 on task 146822, dataset 'segment'.
[2018-04-11 20:51:20] Error in run 956: Run already exists in server. Run id(s): {8961337}
[2018-04-11 20:51:20] Started run 964 on task 146822, dataset 'segment'.
[2018-04-11 20:51:20] Error in run 957: Run already exists in server. Run id(s): {8961339}
[2018-04-11 20:51:20] Started run 965 on task 146822, dataset 'segment'.
[2018-04-11 20:51:21] Error in run 958: Run already exists in server. Run id(s): {8961343}
[2018-04-11 20:51:21] Started run 966 on task 146822, dataset 'segment'.
[2018-04-11 20:51:21] Error in run 959: Run already exists in server. Run id(s): {8961341}
[2018-04-11 20:51:21] Started run 967 on task 146822, dataset 'segment'.
[2018-04-11 20:51:22] Error in run 960: Run already exists in server. Run id(s): {8961344}
[2018-04-11 20:51:22] Started run 968 on task 146822, dataset 'segment'.
[2018-04-11 20:51:24] Error in run 961: Run already exists in server. Run id(s): {8961345}
[2018-04-11 20:51:24] Started run 969 on task 146822, dataset 'segment'.
[2018-04-11 20:51:24] Error in run 962: Run already exists in server. Run id(s): {8961347}
[2018-04-11 20:51:24] Started run 970 on task 146822, dataset 'segment'.
[2018-04-11 20:51:24] Error in run 963: Run already exists in server. Run id(s): {8961348}
[2018-04-11 20:51:24] Started run 971 on task 146822, dataset 'segment'.
[2018-04-11 20:51:25] Error in run 964: Run already exists in server. Run id(s): {8961349}
[2018-04-11 20:51:25] Started run 972 on task 146822, dataset 'segment'.
[2018-04-11 20:51:25] Error in run 965: Run already exists in server. Run id(s): {8961350}
[2018-04-11 20:51:25] Started run 973 on task 146822, dataset 'segment'.
[2018-04-11 20:51:25] Error in run 966: Run already exists in server. Run id(s): {8961353}
[2018-04-11 20:51:25] Started run 974 on task 146822, dataset 'segment'.
[2018-04-11 20:51:26] Error in run 967: Run already exists in server. Run id(s): {8961355}
[2018-04-11 20:51:26] Started run 975 on task 146822, dataset 'segment'.
[2018-04-11 20:51:27] Error in run 968: Run already exists in server. Run id(s): {8961356}
[2018-04-11 20:51:27] Started run 976 on task 146822, dataset 'segment'.
[2018-04-11 20:51:28] Error in run 969: Run already exists in server. Run id(s): {8961357}
[2018-04-11 20:51:28] Started run 977 on task 146822, dataset 'segment'.
[2018-04-11 20:51:29] Error in run 970: Run already exists in server. Run id(s): {8961358}
[2018-04-11 20:51:29] Started run 978 on task 146822, dataset 'segment'.
[2018-04-11 20:51:29] Error in run 971: Run already exists in server. Run id(s): {8961360}
[2018-04-11 20:51:29] Error in run 972: Run already exists in server. Run id(s): {8961364}
[2018-04-11 20:51:29] Started run 979 on task 146822, dataset 'segment'.
[2018-04-11 20:51:29] Started run 980 on task 146822, dataset 'segment'.
[2018-04-11 20:51:29] Error in run 973: Run already exists in server. Run id(s): {8961362}
[2018-04-11 20:51:29] Started run 981 on task 146822, dataset 'segment'.
[2018-04-11 20:51:30] Error in run 974: Run already exists in server. Run id(s): {8961363}
[2018-04-11 20:51:30] Started run 982 on task 146822, dataset 'segment'.
[2018-04-11 20:51:30] Error in run 975: Run already exists in server. Run id(s): {8961366}
[2018-04-11 20:51:30] Started run 983 on task 146822, dataset 'segment'.
[2018-04-11 20:51:31] Error in run 976: Run already exists in server. Run id(s): {8961368}
[2018-04-11 20:51:31] Started run 984 on task 146822, dataset 'segment'.
[2018-04-11 20:51:33] Error in run 977: Run already exists in server. Run id(s): {8961369}
[2018-04-11 20:51:33] Started run 985 on task 146822, dataset 'segment'.
[2018-04-11 20:51:33] Error in run 978: Run already exists in server. Run id(s): {8961370}
[2018-04-11 20:51:33] Started run 986 on task 146822, dataset 'segment'.
[2018-04-11 20:51:34] Error in run 979: Run already exists in server. Run id(s): {8961372}
[2018-04-11 20:51:34] Started run 987 on task 146822, dataset 'segment'.
[2018-04-11 20:51:34] Error in run 980: Run already exists in server. Run id(s): {8961376}
[2018-04-11 20:51:34] Started run 988 on task 146822, dataset 'segment'.
[2018-04-11 20:51:34] Error in run 981: Run already exists in server. Run id(s): {8961373}
[2018-04-11 20:51:34] Started run 989 on task 146822, dataset 'segment'.
[2018-04-11 20:51:34] Error in run 982: Run already exists in server. Run id(s): {8961375}
[2018-04-11 20:51:34] Started run 990 on task 146822, dataset 'segment'.
[2018-04-11 20:51:35] Error in run 983: Run already exists in server. Run id(s): {8961379}
[2018-04-11 20:51:35] Started run 991 on task 146822, dataset 'segment'.
[2018-04-11 20:51:36] Error in run 984: Run already exists in server. Run id(s): {8961381}
[2018-04-11 20:51:36] Started run 992 on task 146822, dataset 'segment'.
[2018-04-11 20:51:38] Error in run 985: Run already exists in server. Run id(s): {8961380}
[2018-04-11 20:51:38] Started run 993 on task 146822, dataset 'segment'.
[2018-04-11 20:51:38] Error in run 986: Run already exists in server. Run id(s): {8961382}
[2018-04-11 20:51:38] Started run 994 on task 146822, dataset 'segment'.
[2018-04-11 20:51:38] Error in run 987: Run already exists in server. Run id(s): {8961384}
[2018-04-11 20:51:38] Started run 995 on task 146822, dataset 'segment'.
[2018-04-11 20:51:38] Error in run 988: Run already exists in server. Run id(s): {8961385}
[2018-04-11 20:51:38] Started run 996 on task 146822, dataset 'segment'.
[2018-04-11 20:51:38] Error in run 989: Run already exists in server. Run id(s): {8961387}
[2018-04-11 20:51:38] Started run 997 on task 146822, dataset 'segment'.
[2018-04-11 20:51:39] Error in run 990: Run already exists in server. Run id(s): {8961388}
[2018-04-11 20:51:39] Started run 998 on task 146822, dataset 'segment'.
[2018-04-11 20:51:40] Error in run 991: Run already exists in server. Run id(s): {8961390}
[2018-04-11 20:51:40] Started run 999 on task 146822, dataset 'segment'.
[2018-04-11 20:51:40] Error in run 992: Run already exists in server. Run id(s): {8961392}
[2018-04-11 20:51:40] Started run 1000 on task 146822, dataset 'segment'.
[2018-04-11 20:51:42] Error in run 993: Run already exists in server. Run id(s): {8961393}
[2018-04-11 20:51:42] Error in run 994: Run already exists in server. Run id(s): {8961394}
[2018-04-11 20:51:43] Error in run 995: Run already exists in server. Run id(s): {8961396}
[2018-04-11 20:51:43] Error in run 996: Run already exists in server. Run id(s): {8961397}
[2018-04-11 20:51:43] Error in run 997: Run already exists in server. Run id(s): {8961399}
[2018-04-11 20:51:44] Error in run 998: Run already exists in server. Run id(s): {8961400}
[2018-04-11 20:51:44] Error in run 999: Run already exists in server. Run id(s): {8961402}
[2018-04-11 20:51:45] Error in run 1000: Run already exists in server. Run id(s): {8961404}
